Python 2.7.10 (default, Oct 23 2015, 19:19:21) 
[GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.0.59.5)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> Starting...

1. 	      : Loading Data
Extracting MNIST_data/train-images-idx3-ubyte.gz
Extracting MNIST_data/train-labels-idx1-ubyte.gz
Extracting MNIST_data/t10k-images-idx3-ubyte.gz
Extracting MNIST_data/t10k-labels-idx1-ubyte.gz
2. - v 	: Session
2017-12-11 12:23:33.259269: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.2 instructions, but these are available on your machine and could speed up CPU computations.
2017-12-11 12:23:33.259281: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX instructions, but these are available on your machine and could speed up CPU computations.
2017-12-11 12:23:33.259297: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX2 instructions, but these are available on your machine and could speed up CPU computations.
2017-12-11 12:23:33.259304: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use FMA instructions, but these are available on your machine and could speed up CPU computations.
3. 	      : epoch: 0
4. 	      : epoch: 1
5. 	      : epoch: 2
Error: 0.100786
6. 	      : Learnrate: 5.0 // Batchsize: 55
2. - ^ 	: Session  (took  77.95683  sek)
/Library/Python/2.7/site-packages/matplotlib/collections.py:548: FutureWarning: elementwise comparison failed; returning scalar instead, but in the future will perform elementwise comparison
  if self._edgecolors == 'face':
>>> Starting...

1. - v 	: Session
2. 	      : epoch: 0
3. 	      : epoch: 1
4. 	      : epoch: 2
Error: 0.100574
5. 	      : Learnrate: 5.0 // Batchsize: 55
1. - ^ 	: Session  (took  110.18941  sek)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 346, in <module>
NameError: name 'pribs' is not defined
>>> >>> >>> >>> >>> a=np.zeros(500)
>>> a.reshape(25,20)
array([[ 0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.]])
>>> sqrt(500)
22.360679774997898
>>> sqrt(499)
22.338307903688676
>>> sqrt(498)
22.315913604421397
>>> sqrt(497)
22.293496809607955
>>> sqrt(496)
22.271057451320086
>>> sqrt(490)
22.135943621178654
>>> sqrt(480)
21.908902300206645
>>> sqrt(485)
22.02271554554524
>>> sqrt(484)
22.0
>>> Starting...

1. - v 	: Session
2. 	      : epoch: 0
3. 	      : epoch: 1
4. 	      : epoch: 2
Error: 0.100458
5. 	      : Learnrate: 5.0 // Batchsize: 55
1. - ^ 	: Session  (took  107.82354  sek)
>>> Starting...

1. - v 	: Session
2. 	      : epoch: 0
3. 	      : epoch: 1
4. 	      : epoch: 2
Error: 0.156337
5. 	      : Learnrate: 1.0 // Batchsize: 100
1. - ^ 	: Session  (took  58.7639  sek)
>>> >>> >>> Starting...

1. - v 	: Session
2. 	      : epoch: 0
3. 	      : epoch: 1
4. 	      : epoch: 2
Info 		: Error: 0.15143 
Info 		: Learnrate: 1.0 // Batchsize: 100 
1. - ^ 	: Session  (took  57.89249  sek)
>>> Starting...

1. - v 	: Session
2. 	      : epoch: 0
3. 	      : epoch: 1
4. 	      : epoch: 2
Info 		: Error: 0.0771562 
Info 		: Learnrate: 10.0 // Batchsize: 100 
1. - ^ 	: Session  (took  133.06193  sek)
>>> Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 2
    map3=plt.matshow(ubv.reshape(28,28))
    ^
IndentationError: unexpected indent
>>> >>> >>> Starting...

1. - v 	: Session Mon Dec 11 12:59:33 2017
2. -- v 	: epoch: 0
2. -- ^ 	: epoch: 0  (took  19.75171  sek)
3. -- v 	: epoch: 1
3. -- ^ 	: epoch: 1  (took  19.33908  sek)
4. -- v 	: epoch: 2
4. -- ^ 	: epoch: 2  (took  19.28539  sek)
Info 		: Error: 0.15 
Info 		: Learnrate: 10.0 // Batchsize: 550 
1. - ^ 	: Session Mon Dec 11 12:59:33 2017  (took  58.76661  sek)
>>> Starting...

1. - v 	: Session from Mon Dec 11 13:01:57 2017
2. -- v 	: epoch: 0
Info 		: Error 0.163966 
2. -- ^ 	: epoch: 0  (took  21.22131  sek)
3. -- v 	: epoch: 1
Info 		: Error 0.158271 
3. -- ^ 	: epoch: 1  (took  19.78616  sek)
4. -- v 	: epoch: 2
Info 		: Error 0.149633 
4. -- ^ 	: epoch: 2  (took  15.66158  sek)
Info 		: Error: 0.149633 
Info 		: Learnrate: 10.0 // Batchsize: 550 
1. - ^ 	: Session from Mon Dec 11 13:01:57 2017  (took  57.00336  sek)
>>> Starting...

1. - v 	: Session
Info 		: Mon Dec 11 13:05:39 2017 
2. -- v 	: epoch: 0
Info 		: Error 0.0822246 
2. -- ^ 	: epoch: 0  (took  46.78547  sek)
3. -- v 	: epoch: 1
Info 		: Error 0.0675703 
3. -- ^ 	: epoch: 1  (took  33.95275  sek)
4. -- v 	: epoch: 2
Info 		: Error 0.0609853 
4. -- ^ 	: epoch: 2  (took  33.3928  sek)
Info 		: Error: 0.0609853 
Info 		: Learnrate: 20.0 // Batchsize: 100 
1. - ^ 	: Session  (took  114.70706  sek)
>>> >>> >>> Starting...

1. - v 	: Session
Info 		: Mon Dec 11 13:15:01 2017 
2. -- v 	: epoch: 0
Info 		: Error 0.163397 
2. -- ^ 	: epoch: 0  (took  50.14776  sek)
3. -- v 	: epoch: 1
Info 		: Error 0.155499 
3. -- ^ 	: epoch: 1  (took  36.24148  sek)
4. -- v 	: epoch: 2
Info 		: Error 0.154087 
4. -- ^ 	: epoch: 2  (took  34.50666  sek)
Info 		: Error: 0.154087 
Info 		: Learnrate: 1.0 // Batchsize: 100 
1. - ^ 	: Session  (took  121.25841  sek)
>>> Starting...

1. - v 	: Session
Info 		: Mon Dec 11 13:18:42 2017 
2. -- v 	: epoch: 0
Info 		: Error 0.16458 
2. -- ^ 	: epoch: 0  (took  94.53318  sek)
3. -- v 	: epoch: 1
Info 		: Error 0.157004 
3. -- ^ 	: epoch: 1  (took  34.00924  sek)
4. -- v 	: epoch: 2
Info 		: Error 0.154369 
4. -- ^ 	: epoch: 2  (took  34.78535  sek)
Info 		: Error: 0.154369 
Info 		: Learnrate: 1.0 // Batchsize: 100 
1. - ^ 	: Session  (took  163.97829  sek)
>>> Starting...

1. - v 	: Session
Info 		: Mon Dec 11 13:22:06 2017 
2. -- v 	: epoch: 0
Info 		: Error 0.154503 
2. -- ^ 	: epoch: 0  (took  47.52908  sek)
Info 		: Error: 0.154503 
Info 		: Learnrate: 1.0 // Batchsize: 100 
1. - ^ 	: Session  (took  47.98405  sek)
>>> Starting...

1. - v 	: Session
Info 		: Mon Dec 11 13:23:04 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.154599 
2. -- ^ 	: epoch: 0  (took  53.84766  sek)
Info 		: Error: 0.154599 
Info 		: Learnrate: 1.0 // Batchsize: 100 
1. - ^ 	: Session  (took  54.29023  sek)
>>> numpoints
<tf.Tensor 'Cast_12:0' shape=() dtype=float32>
>>> numpoints.eval()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/ops.py", line 541, in eval
    return _eval_using_default_session(self, feed_dict, self.graph, session)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/ops.py", line 4071, in _eval_using_default_session
    raise ValueError("Cannot evaluate tensor using `eval()`: No default "
ValueError: Cannot evaluate tensor using `eval()`: No default session is registered. Use `with sess.as_default()` or pass an explicit session to `eval(session=sess)`
>>> numpoints.eval(session=sess)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/ops.py", line 541, in eval
    return _eval_using_default_session(self, feed_dict, self.graph, session)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/ops.py", line 4085, in _eval_using_default_session
    return session.run(tensors, feed_dict)
  File "/Library/Python/2.7/site-packages/tensorflow/python/client/session.py", line 895, in run
    run_metadata_ptr)
  File "/Library/Python/2.7/site-packages/tensorflow/python/client/session.py", line 1051, in _run
    raise RuntimeError('Attempted to use a closed Session.')
RuntimeError: Attempted to use a closed Session.
>>> Starting...

1. - v 	: Session
Info 		: Mon Dec 11 13:25:16 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.10247 
2. -- ^ 	: epoch: 0  (took  48.93077  sek)
Info 		: Error: 0.10247 
Info 		: Learnrate: 1.0 // Batchsize: 100 
1. - ^ 	: Session  (took  49.68859  sek)
>>> Starting...

1. - v 	: Session
Info 		: Mon Dec 11 13:26:53 2017 
2. -- v 	: epoch: 0
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 289, in <module>
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/ops.py", line 541, in eval
    return _eval_using_default_session(self, feed_dict, self.graph, session)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/ops.py", line 4085, in _eval_using_default_session
    return session.run(tensors, feed_dict)
  File "/Library/Python/2.7/site-packages/tensorflow/python/client/session.py", line 895, in run
    run_metadata_ptr)
  File "/Library/Python/2.7/site-packages/tensorflow/python/client/session.py", line 1124, in _run
    feed_dict_tensor, options, run_metadata)
  File "/Library/Python/2.7/site-packages/tensorflow/python/client/session.py", line 1321, in _do_run
    options, run_metadata)
  File "/Library/Python/2.7/site-packages/tensorflow/python/client/session.py", line 1340, in _do_call
    raise type(e)(node_def, op, message)
tensorflow.python.framework.errors_impl.InvalidArgumentError: You must feed a value for placeholder tensor 'Visible-Layer_14' with dtype float and shape [?,784]
	 [[Node: Visible-Layer_14 = Placeholder[dtype=DT_FLOAT, shape=[?,784], _device="/job:localhost/replica:0/task:0/cpu:0"]()]]

Caused by op u'Visible-Layer_14', defined at:
  File "<stdin>", line 1, in <module>
  File "<string>", line 220, in <module>
  File "/Library/Python/2.7/site-packages/tensorflow/python/ops/array_ops.py", line 1548, in placeholder
    return gen_array_ops._placeholder(dtype=dtype, shape=shape, name=name)
  File "/Library/Python/2.7/site-packages/tensorflow/python/ops/gen_array_ops.py", line 2094, in _placeholder
    name=name)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/op_def_library.py", line 767, in apply_op
    op_def=op_def)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/ops.py", line 2630, in create_op
    original_op=self._default_original_op, op_def=op_def)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/ops.py", line 1204, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

InvalidArgumentError (see above for traceback): You must feed a value for placeholder tensor 'Visible-Layer_14' with dtype float and shape [?,784]
	 [[Node: Visible-Layer_14 = Placeholder[dtype=DT_FLOAT, shape=[?,784], _device="/job:localhost/replica:0/task:0/cpu:0"]()]]

>>> Starting...

1. - v 	: Session
Info 		: Mon Dec 11 13:27:07 2017 
2. -- v 	: epoch: 0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0
100.0

***RepPython 2.7.10 (default, Oct 23 2015, 19:19:21) 
[GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.0.59.5)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> Starting...

1. 	      : Loading Data
Extracting MNIST_data/train-images-idx3-ubyte.gz
Extracting MNIST_data/train-labels-idx1-ubyte.gz
Extracting MNIST_data/t10k-images-idx3-ubyte.gz
Extracting MNIST_data/t10k-labels-idx1-ubyte.gz
2. - v 	: Session
Info 		: Mon Dec 11 13:27:54 2017 
2017-12-11 13:27:54.570936: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.2 instructions, but these are available on your machine and could speed up CPU computations.
2017-12-11 13:27:54.570956: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX instructions, but these are available on your machine and could speed up CPU computations.
2017-12-11 13:27:54.570973: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX2 instructions, but these are available on your machine and could speed up CPU computations.
2017-12-11 13:27:54.570979: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use FMA instructions, but these are available on your machine and could speed up CPU computations.
3. -- v 	: epoch: 0
78400.0
Info 		: 	Error 0.102688 
3. -- ^ 	: epoch: 0  (took  41.02791  sek)
Info 		: Error: 0.102688 
Info 		: Learnrate: 1.0 // Batchsize: 100 
2. - ^ 	: Session  (took  41.37078  sek)
/Library/Python/2.7/site-packages/matplotlib/collections.py:548: FutureWarning: elementwise comparison failed; returning scalar instead, but in the future will perform elementwise comparison
  if self._edgecolors == 'face':
>>> Starting...

1. - v 	: Session
Info 		: Mon Dec 11 13:29:34 2017 
2. -- v 	: epoch: 0
220.0
Info 		: 	Error 0.131923 
2. -- ^ 	: epoch: 0  (took  28.23252  sek)
Info 		: Error: 0.131923 
Info 		: Learnrate: 1.0 // Batchsize: 220 
1. - ^ 	: Session  (took  28.45464  sek)
>>> Starting...

1. - v 	: Session
Info 		: Mon Dec 11 13:32:51 2017 
2. -- v 	: epoch: 0
550.0
Info 		: 	Error 0.0977916 
2. -- ^ 	: epoch: 0  (took  20.87962  sek)
Info 		: Error: 0.0977916 
Info 		: Learnrate: 1.0 // Batchsize: 550 
1. - ^ 	: Session  (took  21.16943  sek)
>>> Starting...

1. - v 	: Session
Info 		: Mon Dec 11 13:33:40 2017 
2. -- v 	: epoch: 0
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 313, in <module>
AttributeError: 'float' object has no attribute 'eval'
>>> Starting...

1. - v 	: Session
Info 		: Mon Dec 11 13:50:47 2017 
2. -- v 	: epoch: 0
550.0
Info 		: 	Error 0.0982396 
2. -- ^ 	: epoch: 0  (took  21.5305  sek)
Info 		: Error: 0.0982396 
Info 		: Learnrate: 1.0 // Batchsize: 550 
1. - ^ 	: Session  (took  22.00602  sek)
>>> Starting...

1. - v 	: Session
Info 		: Mon Dec 11 13:51:44 2017 
2. -- v 	: epoch: 0
550.0
Info 		: 	Error 0.148574 
2. -- ^ 	: epoch: 0  (took  18.98357  sek)
Info 		: Error: 0.148574 
Info 		: Learnrate: 10.0 // Batchsize: 550 
1. - ^ 	: Session  (took  19.30568  sek)
>>> Starting...

1. - v 	: Session
Info 		: Mon Dec 11 13:52:15 2017 
2. -- v 	: epoch: 0
55.0
Info 		: 	Error 0.0598742 
2. -- ^ 	: epoch: 0  (took  77.83353  sek)
Info 		: Error: 0.0598742 
Info 		: Learnrate: 1.0 // Batchsize: 55 
1. - ^ 	: Session  (took  78.14814  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 13:54:53 2017 
2. -- v 	: epoch: 0
1375.0
Info 		: 	Error 0.125536 
2. -- ^ 	: epoch: 0  (took  15.77782  sek)
------------------------------------------------
Info 		: Error: 0.125536 
Info 		: Learnrate: 1.0 // Batchsize: 1375 
1. - ^ 	: Session  (took  16.15317  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 13:55:49 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.0550629 
2. -- ^ 	: epoch: 0  (took  85.59914  sek)
------------------------------------------------
Info 		: Error: 0.0550629 
Info 		: Learnrate: 1.0 // Batchsize: 55 
1. - ^ 	: Session  (took  85.99019  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 13:58:55 2017 
2. -- v 	: epoch: 0
Python 2.7.10 (default, Oct 23 2015, 19:19:21) 
[GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.0.59.5)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> Starting...

1. 	      : Loading Data
Extracting MNIST_data/train-images-idx3-ubyte.gz
Extracting MNIST_data/train-labels-idx1-ubyte.gz
Extracting MNIST_data/t10k-images-idx3-ubyte.gz
Extracting MNIST_data/t10k-labels-idx1-ubyte.gz
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 14:04:37 2017 
2017-12-11 14:04:37.348055: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.2 instructions, but these are available on your machine and could speed up CPU computations.
2017-12-11 14:04:37.348067: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX instructions, but these are available on your machine and could speed up CPU computations.
2017-12-11 14:04:37.348083: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX2 instructions, but these are available on your machine and could speed up CPU computations.
2017-12-11 14:04:37.348089: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use FMA instructions, but these are available on your machine and could speed up CPU computations.
2. -- v 	: epoch: 0
Info 		: 	Error 0.0801329 
2. -- ^ 	: epoch: 0  (took  24.68506  sek)
------------------------------------------------
Info 		: Error: 0.0801329 
Info 		: Learnrate: 2.0 // Batchsize: 55 
1. - ^ 	: Session  (took  24.75392  sek)
/Library/Python/2.7/site-packages/matplotlib/collections.py:548: FutureWarning: elementwise comparison failed; returning scalar instead, but in the future will perform elementwise comparison
  if self._edgecolors == 'face':
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 14:05:36 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.0480488 
2. -- ^ 	: epoch: 0  (took  24.84262  sek)
3. -- v 	: epoch: 1
Info 		: 	Error 0.0420268 
3. -- ^ 	: epoch: 1  (took  24.67158  sek)
4. -- v 	: epoch: 2
Info 		: 	Error 0.0412598 
4. -- ^ 	: epoch: 2  (took  24.80033  sek)
5. -- v 	: epoch: 3
Info 		: 	Error 0.0401483 
5. -- ^ 	: epoch: 3  (took  24.98988  sek)
------------------------------------------------
Info 		: Error: 0.0401483 
Info 		: Learnrate: 0.5 // Batchsize: 55 
1. - ^ 	: Session  (took  99.40026  sek)
>>> >>> saved weights and biases
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 14:11:08 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.0474147 
2. -- ^ 	: epoch: 0  (took  35.80932  sek)
3. -- v 	: epoch: 1
Info 		: 	Error 0.042192 
3. -- ^ 	: epoch: 1  (took  34.45736  sek)
4. -- v 	: epoch: 2
Info 		: 	Error 0.0407346 
4. -- ^ 	: epoch: 2  (took  34.69032  sek)
5. -- v 	: epoch: 3
Info 		: 	Error 0.0393897 
5. -- ^ 	: epoch: 3  (took  35.37598  sek)
------------------------------------------------
Info 		: Error: 0.0393897 
Info 		: Learnrate: 0.5 // Batchsize: 55 
1. - ^ 	: Session  (took  140.53976  sek)
>>> Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 1, in <module>
TypeError: save() takes at most 3 arguments (4 given)
>>> >>> saved weights and biases
>>> error_i
0.039389666
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 14:18:35 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.0478937 
2. -- ^ 	: epoch: 0  (took  37.54796  sek)
3. -- v 	: epoch: 1
Info 		: 	Error 0.0430558 
3. -- ^ 	: epoch: 1  (took  36.97706  sek)
4. -- v 	: epoch: 2
Info 		: 	Error 0.0416268 
4. -- ^ 	: epoch: 2  (took  37.41378  sek)
5. -- v 	: epoch: 3
Info 		: 	Error 0.0404403 
5. -- ^ 	: epoch: 3  (took  36.3466  sek)
------------------------------------------------
Info 		: Error: 0.0404403 
Info 		: Learnrate: 0.5 // Batchsize: 55 
1. - ^ 	: Session  (took  148.57959  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 14:22:38 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.0866907 
2. -- ^ 	: epoch: 0  (took  26.68835  sek)
------------------------------------------------
Info 		: Error: 0.0866907 
Info 		: Learnrate: 1.0 // Batchsize: 550 
1. - ^ 	: Session  (took  27.12782  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 14:23:28 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.0951607 
2. -- ^ 	: epoch: 0  (took  26.56549  sek)
------------------------------------------------
Info 		: Error: 0.0951607 
Info 		: Learnrate: 3.0 // Batchsize: 550 
1. - ^ 	: Session  (took  26.8874  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 14:24:04 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.122633 
2. -- ^ 	: epoch: 0  (took  26.13101  sek)
------------------------------------------------
Info 		: Error: 0.122633 
Info 		: Learnrate: 30.0 // Batchsize: 550 
1. - ^ 	: Session  (took  26.44894  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 14:24:41 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.153985 
2. -- ^ 	: epoch: 0  (took  22.90585  sek)
------------------------------------------------
Info 		: Error: 0.153985 
Info 		: Learnrate: 0.1 // Batchsize: 550 
1. - ^ 	: Session  (took  23.2633  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 14:25:12 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.106429 
2. -- ^ 	: epoch: 0  (took  26.42714  sek)
------------------------------------------------
Info 		: Error: 0.106429 
Info 		: Learnrate: 0.01 // Batchsize: 550 
1. - ^ 	: Session  (took  26.82999  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 14:25:59 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.165875 
2. -- ^ 	: epoch: 0  (took  13.92677  sek)
3. -- v 	: epoch: 1
Info 		: 	Error 0.159652 
3. -- ^ 	: epoch: 1  (took  12.90589  sek)
4. -- v 	: epoch: 2
Info 		: 	Error 0.155436 
4. -- ^ 	: epoch: 2  (took  14.63548  sek)
5. -- v 	: epoch: 3
Info 		: 	Error 0.148716 
5. -- ^ 	: epoch: 3  (took  12.95106  sek)
6. -- v 	: epoch: 4
Info 		: 	Error 0.140401 
6. -- ^ 	: epoch: 4  (took  13.37389  sek)
------------------------------------------------
Info 		: Error: 0.140401 
Info 		: Learnrate: 0.01 // Batchsize: 550 
1. - ^ 	: Session  (took  68.15522  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 14:27:23 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.166829 
2. -- ^ 	: epoch: 0  (took  14.26323  sek)
3. -- v 	: epoch: 1
Info 		: 	Error 0.160062 
3. -- ^ 	: epoch: 1  (took  13.81811  sek)
4. -- v 	: epoch: 2
Info 		: 	Error 0.156906 
4. -- ^ 	: epoch: 2  (took  14.18358  sek)
5. -- v 	: epoch: 3
Info 		: 	Error 0.149712 
5. -- ^ 	: epoch: 3  (took  15.07947  sek)
6. -- v 	: epoch: 4
Info 		: 	Error 0.140535 
6. -- ^ 	: epoch: 4  (took  13.24235  sek)
------------------------------------------------
Info 		: Error: 0.140535 
Info 		: Learnrate: 0.1 // Batchsize: 550 
1. - ^ 	: Session  (took  71.00098  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 14:28:48 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.106321 
2. -- ^ 	: epoch: 0  (took  26.42124  sek)
3. -- v 	: epoch: 1
Info 		: 	Error 0.0880975 
3. -- ^ 	: epoch: 1  (took  26.89694  sek)
------------------------------------------------
Info 		: Error: 0.0880975 
Info 		: Learnrate: 0.05 // Batchsize: 550 
1. - ^ 	: Session  (took  53.89892  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 14:30:00 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.132124 
2. -- ^ 	: epoch: 0  (took  26.42152  sek)
3. -- v 	: epoch: 1
Info 		: 	Error 0.106798 
3. -- ^ 	: epoch: 1  (took  26.93999  sek)
------------------------------------------------
Info 		: Error: 0.106798 
Info 		: Learnrate: 0.1 // Batchsize: 550 
1. - ^ 	: Session  (took  53.97476  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 14:31:12 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.106806 
2. -- ^ 	: epoch: 0  (took  26.51815  sek)
3. -- v 	: epoch: 1
Info 		: 	Error 0.0885898 
3. -- ^ 	: epoch: 1  (took  27.72628  sek)
------------------------------------------------
Info 		: Error: 0.0885898 
Info 		: Learnrate: 0.5 // Batchsize: 550 
1. - ^ 	: Session  (took  54.82322  sek)
>>> Starting...
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 215, in <module>
NameError: name 'temp' is not defined
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 14:37:24 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.0578199 
2. -- ^ 	: epoch: 0  (took  59.41128  sek)
3. -- v 	: epoch: 1
Info 		: 	Error 0.0493566 
3. -- ^ 	: epoch: 1  (took  59.5437  sek)
------------------------------------------------
Info 		: Error: 0.0493566 
Info 		: Learnrate: 0.5 // Batchsize: 183 
1. - ^ 	: Session  (took  119.99115  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 14:40:29 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.0990539 
2. -- ^ 	: epoch: 0  (took  45.58405  sek)
------------------------------------------------
Info 		: Error: 0.0990539 
Info 		: Learnrate: 0.5  // Batchsize: 275  // Temp.: 10.0 
1. - ^ 	: Session  (took  46.29445  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 14:41:51 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.150755 
2. -- ^ 	: epoch: 0  (took  45.31378  sek)
------------------------------------------------
Info 		: Error: 0.150755 
Info 		: Learnrate: 0.5  // Batchsize: 110  // Temp.: 100.0 
1. - ^ 	: Session  (took  46.01806  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 14:42:49 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.0550561 
2. -- ^ 	: epoch: 0  (took  49.4214  sek)
------------------------------------------------
Info 		: Error: 0.0550561 
Info 		: Learnrate: 0.5  // Batchsize: 110  // Temp.: 0.5 
1. - ^ 	: Session  (took  50.13501  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 14:44:06 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.0484606 
2. -- ^ 	: epoch: 0  (took  45.82659  sek)
------------------------------------------------
Info 		: Error: 0.0484606 
Info 		: Learnrate: 0.5  // Batchsize: 110  // Temp.: 1.5 
1. - ^ 	: Session  (took  46.61838  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 14:45:44 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.0512226 
2. -- ^ 	: epoch: 0  (took  43.03897  sek)
------------------------------------------------
Info 		: Error: 0.0512226 
Info 		: Learnrate: 1.0  // Batchsize: 110  // Temp.: 1.5 
1. - ^ 	: Session  (took  43.8101  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 14:47:01 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.0504039 
2. -- ^ 	: epoch: 0  (took  19.07121  sek)
------------------------------------------------
Info 		: Error: 0.0504039 
Info 		: Learnrate: 0.5  // Batchsize: 110  // Temp.: 2.5 
1. - ^ 	: Session  (took  19.84258  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 14:48:04 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.053999 
2. -- ^ 	: epoch: 0  (took  9.40357  sek)
------------------------------------------------
Info 		: Error: 0.053999 
Info 		: Learnrate: 0.5  // Batchsize: 110  // Temp.: 1.5 
1. - ^ 	: Session  (took  10.31954  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 14:48:55 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.0595163 
2. -- ^ 	: epoch: 0  (took  8.03535  sek)
3. -- v 	: epoch: 1
Info 		: 	Error 0.0521855 
3. -- ^ 	: epoch: 1  (took  8.0349  sek)
------------------------------------------------
Info 		: Error: 0.0521855 
Info 		: Learnrate: 0.5  // Batchsize: 110  // Temp.: 0.5 
1. - ^ 	: Session  (took  16.88982  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 14:49:44 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.104847 
2. -- ^ 	: epoch: 0  (took  45.56082  sek)
3. -- v 	: epoch: 1
Info 		: 	Error 0.102021 
3. -- ^ 	: epoch: 1  (took  22.99723  sek)
------------------------------------------------
Info 		: Error: 0.102021 
Info 		: Learnrate: 0.5  // Batchsize: 110  // Temp.: 0.1 
1. - ^ 	: Session  (took  69.51055  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 14:51:03 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.05549 
2. -- ^ 	: epoch: 0  (took  51.13471  sek)
3. -- v 	: epoch: 1
Info 		: 	Error 0.0498095 
3. -- ^ 	: epoch: 1  (took  59.05288  sek)
------------------------------------------------
Info 		: Error: 0.0498095 
Info 		: Learnrate: 0.25  // Batchsize: 110  // Temp.: 1.0 
1. - ^ 	: Session  (took  111.23936  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 14:53:30 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.0545011 
2. -- ^ 	: epoch: 0  (took  35.11473  sek)
3. -- v 	: epoch: 1
Info 		: 	Error 0.0503938 
3. -- ^ 	: epoch: 1  (took  24.41672  sek)
------------------------------------------------
Info 		: Error: 0.0503938 
Info 		: Learnrate: 0.25  // Batchsize: 110  // Temp.: 1.0 
1. - ^ 	: Session  (took  60.60416  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 14:55:48 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.0765042 
2. -- ^ 	: epoch: 0  (took  48.69909  sek)
3. -- v 	: epoch: 1
Info 		: 	Error 0.0669592 
3. -- ^ 	: epoch: 1  (took  27.3903  sek)
------------------------------------------------
Info 		: Error: 0.0669592 
Info 		: Learnrate: 0.1  // Batchsize: 110  // Temp.: 1.0 
1. - ^ 	: Session  (took  77.2177  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 14:57:20 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.110943 
2. -- ^ 	: epoch: 0  (took  44.93541  sek)
3. -- v 	: epoch: 1
Info 		: 	Error 0.0987729 
3. -- ^ 	: epoch: 1  (took  28.62128  sek)
------------------------------------------------
Info 		: Error: 0.0987729 
Info 		: Learnrate: 0.1  // Batchsize: 110  // Temp.: 1.0 
1. - ^ 	: Session  (took  74.79624  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 14:58:55 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.166606 
2. -- ^ 	: epoch: 0  (took  24.50315  sek)
3. -- v 	: epoch: 1
Info 		: 	Error 0.153646 
3. -- ^ 	: epoch: 1  (took  17.15029  sek)
------------------------------------------------
Info 		: Error: 0.153646 
Info 		: Learnrate: 0.01  // Batchsize: 110  // Temp.: 1.0 
1. - ^ 	: Session  (took  42.78777  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:01:08 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.0537038 
2. -- ^ 	: epoch: 0  (took  32.19706  sek)
3. -- v 	: epoch: 1
Info 		: 	Error 0.048634 
3. -- ^ 	: epoch: 1  (took  22.8009  sek)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 329, in <module>
TypeError: 'module' object is not callable
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:02:25 2017 
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 329, in <module>
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/ops.py", line 541, in eval
    return _eval_using_default_session(self, feed_dict, self.graph, session)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/ops.py", line 4085, in _eval_using_default_session
    return session.run(tensors, feed_dict)
  File "/Library/Python/2.7/site-packages/tensorflow/python/client/session.py", line 895, in run
    run_metadata_ptr)
  File "/Library/Python/2.7/site-packages/tensorflow/python/client/session.py", line 1093, in _run
    np_val = np.asarray(subfeed_val, dtype=subfeed_dtype)
  File "/Library/Python/2.7/site-packages/numpy/core/numeric.py", line 531, in asarray
    return array(a, dtype, copy=False, order=order)
ValueError: setting an array element with a sequence.
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:03:24 2017 
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 332, in <module>
TypeError: 'module' object is not callable
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:03:37 2017 
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 332, in <module>
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/ops.py", line 541, in eval
    return _eval_using_default_session(self, feed_dict, self.graph, session)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/ops.py", line 4085, in _eval_using_default_session
    return session.run(tensors, feed_dict)
  File "/Library/Python/2.7/site-packages/tensorflow/python/client/session.py", line 895, in run
    run_metadata_ptr)
  File "/Library/Python/2.7/site-packages/tensorflow/python/client/session.py", line 1100, in _run
    % (np_val.shape, subfeed_t.name, str(subfeed_t.get_shape())))
ValueError: Cannot feed value of shape (784,) for Tensor u'Visible-Layer_33:0', which has shape '(?, 784)'
>>> Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 332
    random_recon=v_recon.eval({v:rnd.random(,784)})
                                            ^
SyntaxError: invalid syntax
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:03:49 2017 
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 332, in <module>
  File "mtrand.pyx", line 814, in mtrand.RandomState.random_sample
TypeError: random_sample() takes at most 1 positional argument (2 given)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:04:12 2017 
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 332, in <module>
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/ops.py", line 541, in eval
    return _eval_using_default_session(self, feed_dict, self.graph, session)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/ops.py", line 4085, in _eval_using_default_session
    return session.run(tensors, feed_dict)
  File "/Library/Python/2.7/site-packages/tensorflow/python/client/session.py", line 895, in run
    run_metadata_ptr)
  File "/Library/Python/2.7/site-packages/tensorflow/python/client/session.py", line 1100, in _run
    % (np_val.shape, subfeed_t.name, str(subfeed_t.get_shape())))
ValueError: Cannot feed value of shape (784,) for Tensor u'Visible-Layer_35:0', which has shape '(?, 784)'
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:04:30 2017 
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 332, in <module>
  File "mtrand.pyx", line 814, in mtrand.RandomState.random_sample
TypeError: random_sample() takes at most 1 positional argument (2 given)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:04:39 2017 
1. - ^ 	: Session  (took  2.09814  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:04:56 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.0565942 
2. -- ^ 	: epoch: 0  (took  9.07665  sek)
3. -- v 	: epoch: 1
Info 		: 	Error 0.0488918 
3. -- ^ 	: epoch: 1  (took  8.46751  sek)
------------------------------------------------
Info 		: Error: 0.0488918 
Info 		: Learnrate: 0.25  // Batchsize: 110  // Temp.: 1.0 
1. - ^ 	: Session  (took  19.25898  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:06:14 2017 
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 290, in <module>
  File "<string>", line 192, in init_pretrained
  File "/Library/Python/2.7/site-packages/numpy/lib/npyio.py", line 896, in loadtxt
    fh = iter(open(fname, 'U'))
IOError: [Errno 2] No such file or directory: 'weights-0.039389666.txt'
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:07:00 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 291, in <module>
  File "/Library/Python/2.7/site-packages/tensorflow/python/ops/variables.py", line 527, in assign
    return state_ops.assign(self._variable, value, use_locking=use_locking)
  File "/Library/Python/2.7/site-packages/tensorflow/python/ops/state_ops.py", line 274, in assign
    validate_shape=validate_shape)
  File "/Library/Python/2.7/site-packages/tensorflow/python/ops/gen_state_ops.py", line 43, in assign
    use_locking=use_locking, name=name)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/op_def_library.py", line 767, in apply_op
    op_def=op_def)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/ops.py", line 2632, in create_op
    set_shapes_for_outputs(ret)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/ops.py", line 1911, in set_shapes_for_outputs
    shapes = shape_func(op)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/ops.py", line 1861, in call_with_requiring
    return call_cpp_shape_fn(op, require_shape_fn=True)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/common_shapes.py", line 595, in call_cpp_shape_fn
    require_shape_fn)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/common_shapes.py", line 659, in _call_cpp_shape_fn_impl
    raise ValueError(err.message)
ValueError: Dimension 1 in both shapes must be equal, but are 200 and 500 for 'Assign_120' (op: 'Assign') with input shapes: [784,200], [784,500].
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:07:14 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 292, in <module>
  File "/Library/Python/2.7/site-packages/tensorflow/python/ops/variables.py", line 527, in assign
    return state_ops.assign(self._variable, value, use_locking=use_locking)
  File "/Library/Python/2.7/site-packages/tensorflow/python/ops/state_ops.py", line 274, in assign
    validate_shape=validate_shape)
  File "/Library/Python/2.7/site-packages/tensorflow/python/ops/gen_state_ops.py", line 43, in assign
    use_locking=use_locking, name=name)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/op_def_library.py", line 767, in apply_op
    op_def=op_def)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/ops.py", line 2632, in create_op
    set_shapes_for_outputs(ret)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/ops.py", line 1911, in set_shapes_for_outputs
    shapes = shape_func(op)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/ops.py", line 1861, in call_with_requiring
    return call_cpp_shape_fn(op, require_shape_fn=True)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/common_shapes.py", line 595, in call_cpp_shape_fn
    require_shape_fn)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/common_shapes.py", line 659, in _call_cpp_shape_fn_impl
    raise ValueError(err.message)
ValueError: Dimension 0 in both shapes must be equal, but are 784 and 500 for 'Assign_125' (op: 'Assign') with input shapes: [784], [500].
>>> init_pretrained(bias_v=1)[0].shape
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
(500,)
>>> init_pretrained(bias_h=1)[0].shape
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
(784,)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:09:17 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
1. - ^ 	: Session  (took  4.36521  sek)
>>> Starting...
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 78, in <module>
  File "RBM_Functions.py", line 155, in sigmoid
    return 1./(1.+tf.exp(-1./T*x))
NameError: global name 'tf' is not defined
>>> Starting...
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 78, in <module>
  File "RBM_Functions.py", line 155, in sigmoid
    def sigmoid(x,T):
NameError: global name 'tf' is not defined
>>> Starting...
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 79, in <module>
  File "RBM_Functions.py", line 155, in sigmoid
    def sigmoid(x,T):
NameError: global name 'tf' is not defined
>>> tf
<module 'tensorflow' from '/Library/Python/2.7/site-packages/tensorflow/__init__.pyc'>
>>> >>> Starting...
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 79, in <module>
  File "RBM_Functions.py", line 155, in sigmoid
    def sigmoid(x,T):
NameError: global name 'tf' is not defined
>>> 
***Repl Killed***

#############
## RESTARPython 2.7.10 (default, Oct 23 2015, 19:19:21) 
[GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.0.59.5)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> Starting...

1. 	      : Loading Data
Extracting MNIST_data/train-images-idx3-ubyte.gz
Extracting MNIST_data/train-labels-idx1-ubyte.gz
Extracting MNIST_data/t10k-images-idx3-ubyte.gz
Extracting MNIST_data/t10k-labels-idx1-ubyte.gz
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:17:02 2017 
2017-12-11 15:17:02.242486: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.2 instructions, but these are available on your machine and could speed up CPU computations.
2017-12-11 15:17:02.242505: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX instructions, but these are available on your machine and could speed up CPU computations.
2017-12-11 15:17:02.242521: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX2 instructions, but these are available on your machine and could speed up CPU computations.
2017-12-11 15:17:02.242527: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use FMA instructions, but these are available on your machine and could speed up CPU computations.
2. -- v 	: epoch: 0
Info 		: 	Error 0.0504139 
2. -- ^ 	: epoch: 0  (took  19.9129  sek)
------------------------------------------------
Info 		: Error: 0.0504139 
Info 		: Learnrate: 0.25  // Batchsize: 110  // Temp.: 1.0 
1. - ^ 	: Session  (took  19.99769  sek)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 197, in <module>
  File "RBM_Functions.py", line 102, in tile_raster_images
    out_array = np.zeros(out_shape, dtype=dt)
NameError: global name 'np' is not defined
>>> >>> 
***Repl Killed***

#############
## RESTARPython 2.7.10 (default, Oct 23 2015, 19:19:21) 
[GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.0.59.5)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> Starting...

1. 	      : Loading Data
Extracting MNIST_data/train-images-idx3-ubyte.gz
Extracting MNIST_data/train-labels-idx1-ubyte.gz
Extracting MNIST_data/t10k-images-idx3-ubyte.gz
Extracting MNIST_data/t10k-labels-idx1-ubyte.gz
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:19:39 2017 
2017-12-11 15:19:39.736901: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.2 instructions, but these are available on your machine and could speed up CPU computations.
2017-12-11 15:19:39.736927: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX instructions, but these are available on your machine and could speed up CPU computations.
2017-12-11 15:19:39.736932: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX2 instructions, but these are available on your machine and could speed up CPU computations.
2017-12-11 15:19:39.736939: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use FMA instructions, but these are available on your machine and could speed up CPU computations.
2. -- v 	: epoch: 0
Info 		: 	Error 0.0492126 
2. -- ^ 	: epoch: 0  (took  19.28121  sek)
------------------------------------------------
Info 		: Error: 0.0492126 
Info 		: Learnrate: 0.25  // Batchsize: 110  // Temp.: 1.0 
1. - ^ 	: Session  (took  19.37206  sek)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 197, in <module>
  File "RBM_Functions.py", line 102, in tile_raster_images
    out_array = np.zeros(out_shape, dtype=dt)
NameError: global name 'np' is not defined
>>> 
***Repl Killed***

#############
## RESTARPython 2.7.10 (default, Oct 23 2015, 19:19:21) 
[GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.0.59.5)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> Starting...

1. 	      : Loading Data
Extracting MNIST_data/train-images-idx3-ubyte.gz
Extracting MNIST_data/train-labels-idx1-ubyte.gz
Extracting MNIST_data/t10k-images-idx3-ubyte.gz
Extracting MNIST_data/t10k-labels-idx1-ubyte.gz
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:20:49 2017 
2017-12-11 15:20:49.570166: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.2 instructions, but these are available on your machine and could speed up CPU computations.
2017-12-11 15:20:49.570181: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX instructions, but these are available on your machine and could speed up CPU computations.
2017-12-11 15:20:49.570190: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX2 instructions, but these are available on your machine and could speed up CPU computations.
2017-12-11 15:20:49.570199: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use FMA instructions, but these are available on your machine and could speed up CPU computations.
2. -- v 	: epoch: 0
Info 		: 	Error 0.049566 
2. -- ^ 	: epoch: 0  (took  20.2568  sek)
------------------------------------------------
Info 		: Error: 0.049566 
Info 		: Learnrate: 0.25  // Batchsize: 110  // Temp.: 1.0 
1. - ^ 	: Session  (took  20.33516  sek)
/Library/Python/2.7/site-packages/matplotlib/collections.py:548: FutureWarning: elementwise comparison failed; returning scalar instead, but in the future will perform elementwise comparison
  if self._edgecolors == 'face':
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:21:52 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.0541772 
2. -- ^ 	: epoch: 0  (took  44.37597  sek)
------------------------------------------------
Info 		: Error: 0.0541772 
Info 		: Learnrate: 0.25  // Batchsize: 110  // Temp.: 1.0 
1. - ^ 	: Session  (took  44.54198  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:24:21 2017 
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 135, in <module>
  File "RBM_Functions.py", line 142, in init_pretrained
    os.chdir(path)
NameError: global name 'os' is not defined
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:24:35 2017 
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 135, in <module>
  File "RBM_Functions.py", line 142, in init_pretrained
    os.chdir(path)
NameError: global name 'os' is not defined
>>> 
***Repl Killed***

#############
## RESTARPython 2.7.10 (default, Oct 23 2015, 19:19:21) 
[GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.0.59.5)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> Starting...

1. 	      : Loading Data
Extracting MNIST_data/train-images-idx3-ubyte.gz
Extracting MNIST_data/train-labels-idx1-ubyte.gz
Extracting MNIST_data/t10k-images-idx3-ubyte.gz
Extracting MNIST_data/t10k-labels-idx1-ubyte.gz
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:25:09 2017 
2017-12-11 15:25:09.410992: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.2 instructions, but these are available on your machine and could speed up CPU computations.
2017-12-11 15:25:09.411005: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX instructions, but these are available on your machine and could speed up CPU computations.
2017-12-11 15:25:09.411021: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX2 instructions, but these are available on your machine and could speed up CPU computations.
2017-12-11 15:25:09.411027: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use FMA instructions, but these are available on your machine and could speed up CPU computations.
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 135, in <module>
  File "/Library/Python/2.7/site-packages/tensorflow/python/ops/variables.py", line 527, in assign
    return state_ops.assign(self._variable, value, use_locking=use_locking)
  File "/Library/Python/2.7/site-packages/tensorflow/python/ops/state_ops.py", line 274, in assign
    validate_shape=validate_shape)
  File "/Library/Python/2.7/site-packages/tensorflow/python/ops/gen_state_ops.py", line 43, in assign
    use_locking=use_locking, name=name)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/op_def_library.py", line 767, in apply_op
    op_def=op_def)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/ops.py", line 2632, in create_op
    set_shapes_for_outputs(ret)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/ops.py", line 1911, in set_shapes_for_outputs
    shapes = shape_func(op)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/ops.py", line 1861, in call_with_requiring
    return call_cpp_shape_fn(op, require_shape_fn=True)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/common_shapes.py", line 595, in call_cpp_shape_fn
    require_shape_fn)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/common_shapes.py", line 659, in _call_cpp_shape_fn_impl
    raise ValueError(err.message)
ValueError: Dimension 1 in both shapes must be equal, but are 200 and 500 for 'Assign_3' (op: 'Assign') with input shapes: [784,200], [784,500].
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:26:03 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 135, in <module>
  File "/Library/Python/2.7/site-packages/tensorflow/python/ops/variables.py", line 527, in assign
    return state_ops.assign(self._variable, value, use_locking=use_locking)
  File "/Library/Python/2.7/site-packages/tensorflow/python/ops/state_ops.py", line 274, in assign
    validate_shape=validate_shape)
  File "/Library/Python/2.7/site-packages/tensorflow/python/ops/gen_state_ops.py", line 43, in assign
    use_locking=use_locking, name=name)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/op_def_library.py", line 767, in apply_op
    op_def=op_def)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/ops.py", line 2632, in create_op
    set_shapes_for_outputs(ret)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/ops.py", line 1911, in set_shapes_for_outputs
    shapes = shape_func(op)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/ops.py", line 1861, in call_with_requiring
    return call_cpp_shape_fn(op, require_shape_fn=True)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/common_shapes.py", line 595, in call_cpp_shape_fn
    require_shape_fn)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/common_shapes.py", line 659, in _call_cpp_shape_fn_impl
    raise ValueError(err.message)
ValueError: Dimension 1 in both shapes must be equal, but are 200 and 500 for 'Assign_7' (op: 'Assign') with input shapes: [784,200], [784,500].
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:26:16 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 174, in <module>
ValueError: operands could not be broadcast together with shapes (9,784) (0,784) 
>>> train_data[0:9][400:].shape
(0, 784)
>>> train_data[0:9].shape
(9, 784)
>>> train_data[0:9:400:].shape
  File "<stdin>", line 1
    train_data[0:9:400:].shape
                      ^
SyntaxError: invalid syntax
>>> train_data[0:9:400].shape
(1, 784)
>>> train_data[0:9:].shape
(9, 784)
>>> train_data[0:9,400:].shape
(9, 384)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:27:35 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 174, in <module>
ValueError: operands could not be broadcast together with shapes (9,784) (9,384) 
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:28:19 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
1. - ^ 	: Session  (took  0.96587  sek)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 215, in <module>
NameError: name 'random_recon' is not defined
>>> >>> Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 1, in <module>
NameError: name 'w_i' is not defined
>>> plt.matshow(w.assign(init_pretrained(w=1)[0])
... )
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Library/Python/2.7/site-packages/matplotlib/pyplot.py", line 2217, in matshow
    fig = figure(fignum, figsize=figaspect(A))
  File "/Library/Python/2.7/site-packages/matplotlib/figure.py", line 1649, in figaspect
    nr, nc = arg.shape[:2]
ValueError: need more than 0 values to unpack
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:29:50 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
1. - ^ 	: Session  (took  1.61034  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:30:27 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
1. - ^ 	: Session  (took  1.09544  sek)
>>> half_images.shape
(9, 784)
>>> half_images[0].shape
(784,)
>>> half_images[0,10]
0.0
>>> half_images[0]
array([ 0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.38 ,
        0.376,  0.302,  0.463,  0.239,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.353,  0.541,  0.922,  0.922,  0.922,
        0.922,  0.922,  0.922,  0.984,  0.984,  0.973,  0.996,  0.961,
        0.922,  0.745,  0.082,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.549,  0.984,
        0.996,  0.996,  0.996,  0.996,  0.996,  0.996,  0.996,  0.996,
        0.996,  0.996,  0.996,  0.996,  0.996,  0.996,  0.741,  0.09 ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.886,  0.996,  0.816,  0.78 ,  0.78 ,  0.78 ,
        0.78 ,  0.545,  0.239,  0.239,  0.239,  0.239,  0.239,  0.502,
        0.871,  0.996,  0.996,  0.741,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0
.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ], dtype=float32)
>>> >>> half_images[0,780:]
array([ 0.,  0.,  0.,  0.], dtype=float32)
>>> half_images[0:9,780:]
array([[ 0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.]], dtype=float32)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:31:43 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
1. - ^ 	: Session  (took  1.87784  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:32:03 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
1. - ^ 	: Session  (took  1.2289  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:32:18 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
1. - ^ 	: Session  (took  1.2719  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:32:28 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
1. - ^ 	: Session  (took  1.37759  sek)
>>> >>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:33:00 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
1. - ^ 	: Session  (took  1.53484  sek)
>>> len(half_images[0])
784
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:33:29 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
1. - ^ 	: Session  (took  1.48472  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:33:53 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
1. - ^ 	: Session  (took  1.59832  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:34:03 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
1. - ^ 	: Session  (took  1.60282  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:34:11 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
1. - ^ 	: Session  (took  1.73536  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:35:07 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
1. - ^ 	: Session  (took  10.27018  sek)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 210, in <module>
IndexError: index 8 is out of bounds for axis 0 with size 8
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:35:49 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
1. - ^ 	: Session  (took  2.84124  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:36:02 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
1. - ^ 	: Session  (took  1.88173  sek)
>>> 
***Repl Killed***

#############
## RESTARPython 2.7.10 (default, Oct 23 2015, 19:19:21) 
[GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.0.59.5)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> Starting...

1. 	      : Loading Data
Extracting MNIST_data/train-images-idx3-ubyte.gz
Extracting MNIST_data/train-labels-idx1-ubyte.gz
Extracting MNIST_data/t10k-images-idx3-ubyte.gz
Extracting MNIST_data/t10k-labels-idx1-ubyte.gz
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:36:24 2017 
2017-12-11 15:36:24.163301: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.2 instructions, but these are available on your machine and could speed up CPU computations.
2017-12-11 15:36:24.163325: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX instructions, but these are available on your machine and could speed up CPU computations.
2017-12-11 15:36:24.163336: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX2 instructions, but these are available on your machine and could speed up CPU computations.
2017-12-11 15:36:24.163346: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use FMA instructions, but these are available on your machine and could speed up CPU computations.
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
1. - ^ 	: Session  (took  0.78636  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:37:13 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 174, in <module>
NameError: name 'half_images' is not defined
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:37:30 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 174, in <module>
NameError: name 'half_images' is not defined
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:37:40 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
1. - ^ 	: Session  (took  0.93617  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:37:58 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
1. - ^ 	: Session  (took  0.98166  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:38:16 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
1. - ^ 	: Session  (took  1.08515  sek)
2017-12-11 15:38:35.277 Python[25366:16411309] [QL] file:///Users/Niermann/Library/QuickLook/qlImageSize.qlgenerator/ has no identifier: ignoring
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:39:02 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
1. - ^ 	: Session  (took  1.09426  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:39:37 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
1. - ^ 	: Session  (took  1.17983  sek)
>>> 
***Repl Killed***

#############
## RESTARPython 2.7.10 (default, Oct 23 2015, 19:19:21) 
[GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.0.59.5)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> Starting...

1. 	      : Loading Data
Extracting MNIST_data/train-images-idx3-ubyte.gz
Extracting MNIST_data/train-labels-idx1-ubyte.gz
Extracting MNIST_data/t10k-images-idx3-ubyte.gz
Extracting MNIST_data/t10k-labels-idx1-ubyte.gz
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:40:11 2017 
2017-12-11 15:40:11.955966: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.2 instructions, but these are available on your machine and could speed up CPU computations.
2017-12-11 15:40:11.955984: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX instructions, but these are available on your machine and could speed up CPU computations.
2017-12-11 15:40:11.956000: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX2 instructions, but these are available on your machine and could speed up CPU computations.
2017-12-11 15:40:11.956005: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use FMA instructions, but these are available on your machine and could speed up CPU computations.
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
1. - ^ 	: Session  (took  0.80489  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:41:40 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.0501352 
2. -- ^ 	: epoch: 0  (took  58.01878  sek)
3. -- v 	: epoch: 1
Info 		: 	Error 0.0476089 
3. -- ^ 	: epoch: 1  (took  49.60064  sek)
------------------------------------------------
Info 		: Error: 0.0476089 
Info 		: Learnrate: 0.2  // Batchsize: 110  // Temp.: 1.2 
1. - ^ 	: Session  (took  107.78187  sek)
/Library/Python/2.7/site-packages/matplotlib/collections.py:548: FutureWarning: elementwise comparison failed; returning scalar instead, but in the future will perform elementwise comparison
  if self._edgecolors == 'face':
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 15:46:48 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.838423 
2. -- ^ 	: epoch: 0  (took  40.36685  sek)
3. -- v 	: epoch: 1
Info 		: 	Error 0.838441 
3. -- ^ 	: epoch: 1  (took  31.86026  sek)
------------------------------------------------
Info 		: Error: 0.838441 
Info 		: Learnrate: 0.2  // Batchsize: 110  // Temp.: 1.2 
1. - ^ 	: Session  (took  72.54473  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 21:13:35 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.0522792 
2. -- ^ 	: epoch: 0  (took  65.02091  sek)
3. -- v 	: epoch: 1
Info 		: 	Error 0.0460132 
3. -- ^ 	: epoch: 1  (took  49.73638  sek)
------------------------------------------------
Info 		: Error: 0.0460132 
Info 		: Learnrate: 0.2  // Batchsize: 110  // Temp.: 1.2 
1. - ^ 	: Session  (took  115.33872  sek)
>>> >>> plt.show()
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 21:17:17 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.0933821 
2. -- ^ 	: epoch: 0  (took  58.72447  sek)
3. -- v 	: epoch: 1
Info 		: 	Error 0.076893 
3. -- ^ 	: epoch: 1  (took  49.90853  sek)
------------------------------------------------
Info 		: Error: 0.076893 
Info 		: Learnrate: 0.2  // Batchsize: 110  // Temp.: 10.2 
1. - ^ 	: Session  (took  108.95428  sek)
>>> Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 61
    temp          = 1.2 """Mit temp=10 kann er auch besser verfollseandigen"""
                                                                             ^
SyntaxError: invalid syntax
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 21:22:02 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.0506488 
2. -- ^ 	: epoch: 0  (took  60.23749  sek)
3. -- v 	: epoch: 1
Info 		: 	Error 0.0462015 
3. -- ^ 	: epoch: 1  (took  36.62265  sek)
------------------------------------------------
Info 		: Error: 0.0462015 
Info 		: Learnrate: 0.2  // Batchsize: 110  // Temp.: 1.2 
1. - ^ 	: Session  (took  97.39673  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 21:26:11 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.0933233 
2. -- ^ 	: epoch: 0  (took  59.05458  sek)
3. -- v 	: epoch: 1
Info 		: 	Error 0.0770185 
3. -- ^ 	: epoch: 1  (took  48.89051  sek)
------------------------------------------------
Info 		: Error: 0.0770185 
Info 		: Learnrate: 0.2  // Batchsize: 110  // Temp.: 10.2 
1. - ^ 	: Session  (took  108.33256  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 21:29:13 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.0741172 
2. -- ^ 	: epoch: 0  (took  23.20524  sek)
3. -- v 	: epoch: 1
Info 		: 	Error 0.0619049 
3. -- ^ 	: epoch: 1  (took  22.69872  sek)
4. -- v 	: epoch: 2
Info 		: 	Error 0.0554781 
4. -- ^ 	: epoch: 2  (took  22.54661  sek)
5. -- v 	: epoch: 3
Info 		: 	Error 0.0530322 
5. -- ^ 	: epoch: 3  (took  22.09152  sek)
------------------------------------------------
Info 		: Error: 0.0530322 
Info 		: Learnrate: 0.2  // Batchsize: 55  // Temp.: 5.2 
1. - ^ 	: Session  (took  91.12474  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 21:31:27 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.167745 
2. -- ^ 	: epoch: 0  (took  8.58618  sek)
3. -- v 	: epoch: 1
Info 		: 	Error 0.154181 
3. -- ^ 	: epoch: 1  (took  8.59882  sek)
------------------------------------------------
Info 		: Error: 0.154181 
Info 		: Learnrate: 0.2  // Batchsize: 550  // Temp.: 10.2 
1. - ^ 	: Session  (took  17.58835  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 21:32:11 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.14492 
2. -- ^ 	: epoch: 0  (took  9.48939  sek)
3. -- v 	: epoch: 1
Info 		: 	Error 0.116679 
3. -- ^ 	: epoch: 1  (took  9.82752  sek)
------------------------------------------------
Info 		: Error: 0.116679 
Info 		: Learnrate: 0.2  // Batchsize: 275  // Temp.: 10.2 
1. - ^ 	: Session  (took  19.72348  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 21:34:21 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.144633 
2. -- ^ 	: epoch: 0  (took  14.40855  sek)
3. -- v 	: epoch: 1
Info 		: 	Error 0.116248 
3. -- ^ 	: epoch: 1  (took  13.90936  sek)
4. -- v 	: epoch: 2
Info 		: 	Error 0.103261 
4. -- ^ 	: epoch: 2  (took  14.01913  sek)
5. -- v 	: epoch: 3
Info 		: 	Error 0.0964069 
5. -- ^ 	: epoch: 3  (took  14.16293  sek)
6. -- v 	: epoch: 4
Info 		: 	Error 0.0899006 
6. -- ^ 	: epoch: 4  (took  14.07909  sek)
------------------------------------------------
Info 		: Error: 0.0899006 
Info 		: Learnrate: 0.2  // Batchsize: 275  // Temp.: 10.2 
1. - ^ 	: Session  (took  71.3112  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 21:37:32 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.0638316 
2. -- ^ 	: epoch: 0  (took  42.64188  sek)
3. -- v 	: epoch: 1
Info 		: 	Error 0.0529711 
3. -- ^ 	: epoch: 1  (took  26.32745  sek)
------------------------------------------------
Info 		: Error: 0.0529711 
Info 		: Learnrate: 0.2  // Batchsize: 110  // Temp.: 2.2 
1. - ^ 	: Session  (took  69.83879  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 21:39:23 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.0562198 
2. -- ^ 	: epoch: 0  (took  39.65804  sek)
3. -- v 	: epoch: 1
Info 		: 	Error 0.0497656 
3. -- ^ 	: epoch: 1  (took  24.6317  sek)
------------------------------------------------
Info 		: Error: 0.0497656 
Info 		: Learnrate: 0.2  // Batchsize: 110  // Temp.: 0.2 
1. - ^ 	: Session  (took  64.92336  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 11 21:40:48 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.062405 
2. -- ^ 	: epoch: 0  (took  41.9595  sek)
3. -- v 	: epoch: 1
Info 		: 	Error 0.0535795 
3. -- ^ 	: epoch: 1  (took  27.82558  sek)
------------------------------------------------
Info 		: Error: 0.0535795 
Info 		: Learnrate: 1.0  // Batchsize: 110  // Temp.: 10.2 
1. - ^ 	: Session  (took  70.47747  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 08:25:57 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.0631217 
2. -- ^ 	: epoch: 0  (took  49.57234  sek)
------------------------------------------------
Info 		: Error: 0.0631217 
Info 		: Learnrate: 0.2  // Batchsize: 110  // Temp.: 2.2 
1. - ^ 	: Session  (took  50.93248  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 08:28:14 2017 
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 144, in <module>
NameError: name 'writer' is not defined
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 08:28:30 2017 
2. 	      : Opening Writer Object <tensorflow.python.summary.writer.writer.FileWriter object at 0x12b8844d0>
3. -- v 	: epoch: 0
Info 		: 	Error 0.063432 
3. -- ^ 	: epoch: 0  (took  12.25527  sek)
4. -- v 	: epoch: 1
Info 		: 	Error 0.0541955 
4. -- ^ 	: epoch: 1  (took  11.9551  sek)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 187, in <module>
NameError: name 'sess1' is not defined
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 08:29:12 2017 
2. 	      : Opening Writer Object
3. -- v 	: epoch: 0
Info 		: 	Error 0.062504 
3. -- ^ 	: epoch: 0  (took  17.60093  sek)
4. -- v 	: epoch: 1
Info 		: 	Error 0.0541225 
4. -- ^ 	: epoch: 1  (took  17.54901  sek)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 187, in <module>
  File "/Library/Python/2.7/site-packages/tensorflow/python/client/session.py", line 895, in run
    run_metadata_ptr)
  File "/Library/Python/2.7/site-packages/tensorflow/python/client/session.py", line 1124, in _run
    feed_dict_tensor, options, run_metadata)
  File "/Library/Python/2.7/site-packages/tensorflow/python/client/session.py", line 1321, in _do_run
    options, run_metadata)
  File "/Library/Python/2.7/site-packages/tensorflow/python/client/session.py", line 1340, in _do_call
    raise type(e)(node_def, op, message)
tensorflow.python.framework.errors_impl.InvalidArgumentError: You must feed a value for placeholder tensor 'Visible-Layer_16' with dtype float and shape [?,784]
	 [[Node: Visible-Layer_16 = Placeholder[dtype=DT_FLOAT, shape=[?,784], _device="/job:localhost/replica:0/task:0/cpu:0"]()]]

Caused by op u'Visible-Layer_16', defined at:
  File "<stdin>", line 1, in <module>
  File "<string>", line 75, in <module>
  File "/Library/Python/2.7/site-packages/tensorflow/python/ops/array_ops.py", line 1548, in placeholder
    return gen_array_ops._placeholder(dtype=dtype, shape=shape, name=name)
  File "/Library/Python/2.7/site-packages/tensorflow/python/ops/gen_array_ops.py", line 2094, in _placeholder
    name=name)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/op_def_library.py", line 767, in apply_op
    op_def=op_def)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/ops.py", line 2630, in create_op
    original_op=self._default_original_op, op_def=op_def)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/ops.py", line 1204, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

InvalidArgumentError (see above for traceback): You must feed a value for placeholder tensor 'Visible-Layer_16' with dtype float and shape [?,784]
	 [[Node: Visible-Layer_16 = Placeholder[dtype=DT_FLOAT, shape=[?,784], _device="/job:localhost/replica:0/task:0/cpu:0"]()]]

>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 08:32:01 2017 
2. 	      : Opening Writer Object
3. -- v 	: epoch: 0
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 165, in <module>
  File "/Library/Python/2.7/site-packages/tensorflow/python/client/session.py", line 895, in run
    run_metadata_ptr)
  File "/Library/Python/2.7/site-packages/tensorflow/python/client/session.py", line 1124, in _run
    feed_dict_tensor, options, run_metadata)
  File "/Library/Python/2.7/site-packages/tensorflow/python/client/session.py", line 1321, in _do_run
    options, run_metadata)
  File "/Library/Python/2.7/site-packages/tensorflow/python/client/session.py", line 1340, in _do_call
    raise type(e)(node_def, op, message)
tensorflow.python.framework.errors_impl.InvalidArgumentError: You must feed a value for placeholder tensor 'Visible-Layer_16' with dtype float and shape [?,784]
	 [[Node: Visible-Layer_16 = Placeholder[dtype=DT_FLOAT, shape=[?,784], _device="/job:localhost/replica:0/task:0/cpu:0"]()]]

Caused by op u'Visible-Layer_16', defined at:
  File "<stdin>", line 1, in <module>
  File "<string>", line 75, in <module>
  File "/Library/Python/2.7/site-packages/tensorflow/python/ops/array_ops.py", line 1548, in placeholder
    return gen_array_ops._placeholder(dtype=dtype, shape=shape, name=name)
  File "/Library/Python/2.7/site-packages/tensorflow/python/ops/gen_array_ops.py", line 2094, in _placeholder
    name=name)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/op_def_library.py", line 767, in apply_op
    op_def=op_def)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/ops.py", line 2630, in create_op
    original_op=self._default_original_op, op_def=op_def)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/ops.py", line 1204, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

InvalidArgumentError (see above for traceback): You must feed a value for placeholder tensor 'Visible-Layer_16' with dtype float and shape [?,784]
	 [[Node: Visible-Layer_16 = Placeholder[dtype=DT_FLOAT, shape=[?,784], _device="/job:localhost/replica:0/task:0/cpu:0"]()]]

>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 08:32:40 2017 
2. 	      : Opening Writer Object
3. -- v 	: epoch: 0
Info 		: 	Error 0.0621977 
3. -- ^ 	: epoch: 0  (took  17.91575  sek)
4. -- v 	: epoch: 1
Info 		: 	Error 0.0541894 
4. -- ^ 	: epoch: 1  (took  18.04482  sek)
5. 	      : Finishing Writing to  <tensorflow.python.summary.writer.writer.FileWriter object at 0x120197dd0>
------------------------------------------------
Info 		: Error: 0.0541894 
Info 		: Learnrate: 0.2  // Batchsize: 110  // Temp.: 2.2 
1. - ^ 	: Session  (took  39.71012  sek)
2017-12-12 08:33:37.328 Python[25602:16499940] [QL] file:///Users/Niermann/Library/QuickLook/qlImageSize.qlgenerator/ has no identifier: ignoring
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 08:34:00 2017 
2. 	      : Opening Writer Object
3. -- v 	: epoch: 0
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 162, in <module>
  File "/Library/Python/2.7/site-packages/tensorflow/python/client/session.py", line 895, in run
    run_metadata_ptr)
  File "/Library/Python/2.7/site-packages/tensorflow/python/client/session.py", line 1124, in _run
    feed_dict_tensor, options, run_metadata)
  File "/Library/Python/2.7/site-packages/tensorflow/python/client/session.py", line 1321, in _do_run
    options, run_metadata)
  File "/Library/Python/2.7/site-packages/tensorflow/python/client/session.py", line 1340, in _do_call
    raise type(e)(node_def, op, message)
tensorflow.python.framework.errors_impl.InvalidArgumentError: You must feed a value for placeholder tensor 'Visible-Layer_16' with dtype float and shape [?,784]
	 [[Node: Visible-Layer_16 = Placeholder[dtype=DT_FLOAT, shape=[?,784], _device="/job:localhost/replica:0/task:0/cpu:0"]()]]

Caused by op u'Visible-Layer_16', defined at:
  File "<stdin>", line 1, in <module>
  File "<string>", line 75, in <module>
  File "/Library/Python/2.7/site-packages/tensorflow/python/ops/array_ops.py", line 1548, in placeholder
    return gen_array_ops._placeholder(dtype=dtype, shape=shape, name=name)
  File "/Library/Python/2.7/site-packages/tensorflow/python/ops/gen_array_ops.py", line 2094, in _placeholder
    name=name)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/op_def_library.py", line 767, in apply_op
    op_def=op_def)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/ops.py", line 2630, in create_op
    original_op=self._default_original_op, op_def=op_def)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/ops.py", line 1204, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

InvalidArgumentError (see above for traceback): You must feed a value for placeholder tensor 'Visible-Layer_16' with dtype float and shape [?,784]
	 [[Node: Visible-Layer_16 = Placeholder[dtype=DT_FLOAT, shape=[?,784], _device="/job:localhost/replica:0/task:0/cpu:0"]()]]

>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 08:34:18 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.0634005 
2. -- ^ 	: epoch: 0  (took  11.3162  sek)
------------------------------------------------
Info 		: Error: 0.0634005 
Info 		: Learnrate: 0.2  // Batchsize: 110  // Temp.: 2.2 
1. - ^ 	: Session  (took  12.41522  sek)
>>> 2017-12-12 08:35:39.161 Python[25602:16502257] Communications error: <OS_xpc_error: <error: 0x7fff7c994b90> { count = 1, contents =
	"XPCErrorDescription" => <string: 0x7fff7c994f40> { length = 22, contents = "Connection interrupted" }
}>
Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 09:15:25 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.0630779 
2. -- ^ 	: epoch: 0  (took  19.9042  sek)
------------------------------------------------
Info 		: Error: 0.0630779 
Info 		: Learnrate: 0.2  // Batchsize: 110  // Temp.: 2.2 
1. - ^ 	: Session  (took  21.60132  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 09:16:06 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.0588404 
2. -- ^ 	: epoch: 0  (took  11.41867  sek)
------------------------------------------------
Info 		: Error: 0.0588404 
Info 		: Learnrate: 0.3  // Batchsize: 110  // Temp.: 2.5 
1. - ^ 	: Session  (took  12.39606  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 09:16:38 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.0597618 
2. -- ^ 	: epoch: 0  (took  11.08113  sek)
3. -- v 	: epoch: 1
Info 		: 	Error 0.0508204 
3. -- ^ 	: epoch: 1  (took  11.16556  sek)
------------------------------------------------
Info 		: Error: 0.0508204 
Info 		: Learnrate: 0.3  // Batchsize: 110  // Temp.: 2.5 
1. - ^ 	: Session  (took  23.27951  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 09:17:22 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.106006 
2. -- ^ 	: epoch: 0  (took  8.62442  sek)
3. -- v 	: epoch: 1
Info 		: 	Error 0.0888564 
3. -- ^ 	: epoch: 1  (took  8.56308  sek)
------------------------------------------------
Info 		: Error: 0.0888564 
Info 		: Learnrate: 0.3  // Batchsize: 550  // Temp.: 2.5 
1. - ^ 	: Session  (took  18.24977  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 09:18:12 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.0819486 
2. -- ^ 	: epoch: 0  (took  9.40899  sek)
3. -- v 	: epoch: 1
Info 		: 	Error 0.0679996 
3. -- ^ 	: epoch: 1  (took  9.55388  sek)
4. -- v 	: epoch: 2
Info 		: 	Error 0.061898 
4. -- ^ 	: epoch: 2  (took  9.51841  sek)
------------------------------------------------
Info 		: Error: 0.061898 
Info 		: Learnrate: 0.3  // Batchsize: 275  // Temp.: 2.5 
1. - ^ 	: Session  (took  29.54112  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 09:19:27 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.0695986 
2. -- ^ 	: epoch: 0  (took  9.36801  sek)
3. -- v 	: epoch: 1
Info 		: 	Error 0.0583496 
3. -- ^ 	: epoch: 1  (took  9.59841  sek)
------------------------------------------------
Info 		: Error: 0.0583496 
Info 		: Learnrate: 0.6  // Batchsize: 275  // Temp.: 2.5 
1. - ^ 	: Session  (took  20.09029  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 09:20:19 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.161429 
2. -- ^ 	: epoch: 0  (took  11.26607  sek)
3. -- v 	: epoch: 1
Info 		: 	Error 0.151924 
3. -- ^ 	: epoch: 1  (took  11.10652  sek)
4. -- v 	: epoch: 2
Info 		: 	Error 0.137638 
4. -- ^ 	: epoch: 2  (took  10.76994  sek)
5. -- v 	: epoch: 3
Info 		: 	Error 0.12472 
5. -- ^ 	: epoch: 3  (took  10.74425  sek)
6. -- v 	: epoch: 4
Info 		: 	Error 0.113932 
6. -- ^ 	: epoch: 4  (took  10.74448  sek)
7. -- v 	: epoch: 5
Info 		: 	Error 0.105929 
7. -- ^ 	: epoch: 5  (took  10.86653  sek)
8. -- v 	: epoch: 6
Info 		: 	Error 0.104194 
8. -- ^ 	: epoch: 6  (took  10.80009  sek)
9. -- v 	: epoch: 7
Info 		: 	Error 0.0989965 
9. -- ^ 	: epoch: 7  (took  10.7293  sek)
10. -- v 	: epoch: 8
Info 		: 	Error 0.098187 
10. -- ^ 	: epoch: 8  (took  10.7534  sek)
11. -- v 	: epoch: 9
Info 		: 	Error 0.0952306 
11. -- ^ 	: epoch: 9  (took  10.73939  sek)
------------------------------------------------
Info 		: Error: 0.0952306 
Info 		: Learnrate: 0.01  // Batchsize: 137  // Temp.: 2.5 
1. - ^ 	: Session  (took  109.72899  sek)
2017-12-12 09:23:15.808 Python[25602:16513389] Communications error: <OS_xpc_error: <error: 0x7fff7c994b90> { count = 1, contents =
	"XPCErrorDescription" => <string: 0x7fff7c994f40> { length = 22, contents = "Connection interrupted" }
}>
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 09:23:27 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.148849 
2. -- ^ 	: epoch: 0  (took  11.66137  sek)
3. -- v 	: epoch: 1
Info 		: 	Error 0.119886 
3. -- ^ 	: epoch: 1  (took  11.15453  sek)
4. -- v 	: epoch: 2
Info 		: 	Error 0.104736 
4. -- ^ 	: epoch: 2  (took  12.89222  sek)
5. -- v 	: epoch: 3
Info 		: 	Error 0.0982645 
5. -- ^ 	: epoch: 3  (took  11.62866  sek)
6. -- v 	: epoch: 4
Info 		: 	Error 0.0937702 
6. -- ^ 	: epoch: 4  (took  10.90713  sek)
7. -- v 	: epoch: 5
Info 		: 	Error 0.0885245 
7. -- ^ 	: epoch: 5  (took  11.43183  sek)
8. -- v 	: epoch: 6
Info 		: 	Error 0.0858355 
8. -- ^ 	: epoch: 6  (took  11.16502  sek)
9. -- v 	: epoch: 7
Info 		: 	Error 0.0811195 
9. -- ^ 	: epoch: 7  (took  11.97544  sek)
10. -- v 	: epoch: 8
Info 		: 	Error 0.0790953 
10. -- ^ 	: epoch: 8  (took  12.82601  sek)
11. -- v 	: epoch: 9
Info 		: 	Error 0.076911 
11. -- ^ 	: epoch: 9  (took  11.58373  sek)
------------------------------------------------
Info 		: Error: 0.076911 
Info 		: Learnrate: 0.02  // Batchsize: 137  // Temp.: 2.5 
1. - ^ 	: Session  (took  118.53068  sek)
>>> Starting...
Finished importing
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 47, in <module>
TypeError: sum() takes no arguments (1 given)
>>> Starting...
Finished importing
3
>>> Starting...
Finished importing
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 48, in <module>
  File "<string>", line 44, in sum
NameError: global name 'x' is not defined
>>> Starting...
Finished importing
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 49, in <module>
  File "<string>", line 45, in sum
NameError: global name 'x' is not defined
>>> Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 44
    locals x,y
           ^
SyntaxError: invalid syntax
>>> Starting...
Finished importing
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 50, in <module>
  File "<string>", line 46, in sum
NameError: global name 'x' is not defined
>>> Starting...
Finished importing
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 51, in <module>
  File "<string>", line 47, in sum
NameError: global name 'x' is not defined
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 09:34:12 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.0735776 
2. -- ^ 	: epoch: 0  (took  15.87962  sek)
------------------------------------------------
Info 		: Error: 0.0735776 
Info 		: Learnrate: 0.2  // Batchsize: 137  // Temp.: 2.5 
1. - ^ 	: Session  (took  18.1661  sek)
>>> 
***Repl Killed***

#############
## RESTARPython 2.7.10 (default, Oct 23 2015, 19:19:21) 
[GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.0.59.5)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> Starting...

1. 	      : Loading Data
Extracting MNIST_data/train-images-idx3-ubyte.gz
Extracting MNIST_data/train-labels-idx1-ubyte.gz
Extracting MNIST_data/t10k-images-idx3-ubyte.gz
Extracting MNIST_data/t10k-labels-idx1-ubyte.gz
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 09:36:13 2017 
2017-12-12 09:36:13.151588: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.2 instructions, but these are available on your machine and could speed up CPU computations.
2017-12-12 09:36:13.151600: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX instructions, but these are available on your machine and could speed up CPU computations.
2017-12-12 09:36:13.151616: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX2 instructions, but these are available on your machine and could speed up CPU computations.
2017-12-12 09:36:13.151622: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use FMA instructions, but these are available on your machine and could speed up CPU computations.
2. -- v 	: epoch: 0
Info 		: 	Error 0.0717089 
2. -- ^ 	: epoch: 0  (took  11.55474  sek)
------------------------------------------------
Info 		: Error: 0.0717089 
Info 		: Learnrate: 0.2  // Batchsize: 137  // Temp.: 2.5 
1. - ^ 	: Session  (took  11.62627  sek)
/Library/Python/2.7/site-packages/matplotlib/collections.py:548: FutureWarning: elementwise comparison failed; returning scalar instead, but in the future will perform elementwise comparison
  if self._edgecolors == 'face':
>>> Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 1
    fig_m=plt.figure("Weights",figsize=(6,4))
    ^
IndentationError: unexpected indent
>>> Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 2, in <module>
AttributeError: 'Figure' object has no attribute 'matshow'
>>> >>> >>> Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 1
    fig_m=plt.figure("Weights",figsize=(6,4))
    ^
IndentationError: unexpected indent
>>> >>> >>> Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 1
    fig_m=plt.figure("Weights",figsize=(8,6))
    ^
IndentationError: unexpected indent
>>> >>> >>> >>> >>> import seaborn
>>> >>> >>> plt.plot(errors)
[<matplotlib.lines.Line2D object at 0x11fef7750>]
>>> plt.show()
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 10:18:11 2017 
2. -- v 	: epoch: 0

***Repl Killed***

#############
## RESTARPython 2.7.10 (default, Oct 23 2015, 19:19:21) 
[GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.0.59.5)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> Starting...

1. 	      : Loading Data
Extracting MNIST_data/train-images-idx3-ubyte.gz
Extracting MNIST_data/train-labels-idx1-ubyte.gz
Extracting MNIST_data/t10k-images-idx3-ubyte.gz
Extracting MNIST_data/t10k-labels-idx1-ubyte.gz
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 10:18:37 2017 
2017-12-12 10:18:37.589838: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.2 instructions, but these are available on your machine and could speed up CPU computations.
2017-12-12 10:18:37.589860: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX instructions, but these are available on your machine and could speed up CPU computations.
2017-12-12 10:18:37.589877: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX2 instructions, but these are available on your machine and could speed up CPU computations.
2017-12-12 10:18:37.589883: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use FMA instructions, but these are available on your machine and could speed up CPU computations.
2. -- v 	: epoch: 0
Info 		: 	Error 0.0705669 
2. -- ^ 	: epoch: 0  (took  42.62983  sek)
------------------------------------------------
Info 		: Error: 0.0705669 
Info 		: Learnrate: 0.2  // Batchsize: 137  // Temp.: 2.5 
1. - ^ 	: Session  (took  42.95256  sek)
/Library/Python/2.7/site-packages/matplotlib/collections.py:548: FutureWarning: elementwise comparison failed; returning scalar instead, but in the future will perform elementwise comparison
  if self._edgecolors == 'face':
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 12:41:18 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.0729178 
2. -- ^ 	: epoch: 0  (took  42.8511  sek)
------------------------------------------------
Info 		: Error: 0.0729178 
Info 		: Learnrate: 0.2  // Batchsize: 137  // Temp.: 2.5 
1. - ^ 	: Session  (took  43.51998  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 12:42:31 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.0723804 
2. -- ^ 	: epoch: 0  (took  15.74113  sek)
3. -- v 	: epoch: 1
Info 		: 	Error 0.0606098 
3. -- ^ 	: epoch: 1  (took  13.62537  sek)
4. -- v 	: epoch: 2
Info 		: 	Error 0.054702 
4. -- ^ 	: epoch: 2  (took  12.56156  sek)
------------------------------------------------
Info 		: Error: 0.054702 
Info 		: Learnrate: 0.2  // Batchsize: 137  // Temp.: 2.5 
1. - ^ 	: Session  (took  42.08019  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 12:47:29 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.0725449 
2. -- ^ 	: epoch: 0  (took  20.25672  sek)
------------------------------------------------
Info 		: Error: 0.0725449 
Info 		: Learnrate: 0.1  // Batchsize: 55  // Temp.: 2.5 
1. - ^ 	: Session  (took  20.61861  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 12:48:14 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.0564941 
2. -- ^ 	: epoch: 0  (took  15.66448  sek)
------------------------------------------------
Info 		: Error: 0.0564941 
Info 		: Learnrate: 0.3  // Batchsize: 55  // Temp.: 2.5 
1. - ^ 	: Session  (took  15.88533  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 12:48:55 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.110734 
2. -- ^ 	: epoch: 0  (took  15.09212  sek)
------------------------------------------------
Info 		: Error: 0.110734 
Info 		: Learnrate: 0.1  // Batchsize: 55  // Temp.: 10.5 
1. - ^ 	: Session  (took  15.42162  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 12:49:41 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.109492 
2. -- ^ 	: epoch: 0  (took  14.35292  sek)
3. -- v 	: epoch: 1
Info 		: 	Error 0.0928709 
3. -- ^ 	: epoch: 1  (took  14.31845  sek)
4. -- v 	: epoch: 2
Info 		: 	Error 0.0831021 
4. -- ^ 	: epoch: 2  (took  14.47401  sek)
5. -- v 	: epoch: 3
Info 		: 	Error 0.0758614 
5. -- ^ 	: epoch: 3  (took  14.4969  sek)
6. -- v 	: epoch: 4
Info 		: 	Error 0.0717811 
6. -- ^ 	: epoch: 4  (took  14.39132  sek)
------------------------------------------------
Info 		: Error: 0.0717811 
Info 		: Learnrate: 0.1  // Batchsize: 55  // Temp.: 10.5 
1. - ^ 	: Session  (took  72.31863  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 12:51:24 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.0701442 
2. -- ^ 	: epoch: 0  (took  14.65398  sek)
------------------------------------------------
Info 		: Error: 0.0701442 
Info 		: Learnrate: 0.5  // Batchsize: 55  // Temp.: 1.5 
1. - ^ 	: Session  (took  15.00541  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 12:52:09 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.0720559 
2. -- ^ 	: epoch: 0  (took  14.60377  sek)
3. -- v 	: epoch: 1
Info 		: 	Error 0.0557934 
3. -- ^ 	: epoch: 1  (took  14.96174  sek)
4. -- v 	: epoch: 2
Info 		: 	Error 0.0513412 
4. -- ^ 	: epoch: 2  (took  14.83539  sek)
------------------------------------------------
Info 		: Error: 0.0513412 
Info 		: Learnrate: 0.5  // Batchsize: 55  // Temp.: 1.5 
1. - ^ 	: Session  (took  44.79577  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 12:53:29 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.0878522 
2. -- ^ 	: epoch: 0  (took  38.3314  sek)
------------------------------------------------
Info 		: Error: 0.0878522 
Info 		: Learnrate: 0.1  // Batchsize: 137  // Temp.: 2.5 
1. - ^ 	: Session  (took  38.86551  sek)
>>> ubh.mean()
0.93603212
>>> ubh.var()
0.025177512
>>> sqrt(ubh.var())
0.15867423331434719
>>> Starting...
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 120, in <module>
NameError: name 'shape_of_h' is not defined
>>> Starting...
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 121, in <module>
  File "/Library/Python/2.7/site-packages/tensorflow/python/ops/math_ops.py", line 1844, in matmul
    a, b, transpose_a=transpose_a, transpose_b=transpose_b, name=name)
  File "/Library/Python/2.7/site-packages/tensorflow/python/ops/gen_math_ops.py", line 1289, in _mat_mul
    transpose_b=transpose_b, name=name)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/op_def_library.py", line 767, in apply_op
    op_def=op_def)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/ops.py", line 2632, in create_op
    set_shapes_for_outputs(ret)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/ops.py", line 1911, in set_shapes_for_outputs
    shapes = shape_func(op)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/ops.py", line 1861, in call_with_requiring
    return call_cpp_shape_fn(op, require_shape_fn=True)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/common_shapes.py", line 595, in call_cpp_shape_fn
    require_shape_fn)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/common_shapes.py", line 659, in _call_cpp_shape_fn_impl
    raise ValueError(err.message)
ValueError: Shape must be rank 2 but is rank 1 for 'MatMul_60' (op: 'MatMul') with input shapes: [500], [500,784].
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 14:00:21 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
1. - ^ 	: Session  (took  2.3282  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 14:01:37 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 188, in <module>
AttributeError: 'module' object has no attribute 'radnom'
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 14:01:45 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
1. - ^ 	: Session  (took  2.48375  sek)
>>> >>> >>> >>> >>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 14:03:09 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 190, in <module>
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/ops.py", line 541, in eval
    return _eval_using_default_session(self, feed_dict, self.graph, session)
  File "/Library/Python/2.7/site-packages/tensorflow/python/framework/ops.py", line 4085, in _eval_using_default_session
    return session.run(tensors, feed_dict)
  File "/Library/Python/2.7/site-packages/tensorflow/python/client/session.py", line 895, in run
    run_metadata_ptr)
  File "/Library/Python/2.7/site-packages/tensorflow/python/client/session.py", line 1100, in _run
    % (np_val.shape, subfeed_t.name, str(subfeed_t.get_shape())))
ValueError: Cannot feed value of shape (1, 784) for Tensor u'Reverse-hidden_4:0', which has shape '(?, 500)'
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 14:03:51 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
1. - ^ 	: Session  (took  2.75978  sek)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 233, in <module>
ValueError: cannot reshape array of size 500 into shape (28,28)
>>> >>> >>> >>> >>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 14:05:09 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
1. - ^ 	: Session  (took  4.112  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 14:05:28 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
1. - ^ 	: Session  (took  10.73515  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 14:06:50 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
1. - ^ 	: Session  (took  19.30217  sek)
>>> 20*20
400
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 14:07:32 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 196, in <module>
ValueError: cannot reshape array of size 500 into shape (20,20)
>>> input_vector[:400].shape
(1, 500)
>>> input_vector[0:400].shape
(1, 500)
>>> input_vector[0].shape
(500,)
>>> input_vector[:,:400].shape
(1, 400)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 14:08:11 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
1. - ^ 	: Session  (took  14.68645  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 14:08:35 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
1. - ^ 	: Session  (took  13.79323  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 14:09:00 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
1. - ^ 	: Session  (took  4.14031  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 14:09:11 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
1. - ^ 	: Session  (took  88.18117  sek)
>>> np.round([1.1])
array([ 1.])
>>> np.round([0.1])
array([ 0.])
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 14:11:06 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
1. - ^ 	: Session  (took  6.49516  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 14:11:33 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 188, in <module>
AttributeError: 'module' object has no attribute 'zeros'
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 14:11:38 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
1. - ^ 	: Session  (took  12.56781  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 14:12:17 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
1. - ^ 	: Session  (took  10.23627  sek)
>>> >>> input_vector
array([[ 0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
         0.,  0.,  0.,  0.,  0.,  0.]])
>>> input_vector[0]
array([ 0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
        0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
        0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
        0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
        0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
        0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
        0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
        0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
        0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
        0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
        0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
        0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
        0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
        0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
        0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
        0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
        0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
        0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
        0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
        0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
        0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
        0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
        0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
        0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
        0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
        0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
        0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
        0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
        0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
        0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
        0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
        0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
        0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
        0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
        0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
        0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
        0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
        0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,
        0.,  0.,  0.,  0.,  0.,  0.])
>>> input_vector[0][0]
0.0
>>> input_vector[0][0:10]
array([ 0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.])
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 14:13:03 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
1. - ^ 	: Session  (took  15.1266  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 14:13:30 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
1. - ^ 	: Session  (took  3.0831  sek)
>>> >>> ax2[1].matshow(input_vector[:,:400].reshape(20,20))
<matplotlib.image.AxesImage object at 0x11ea69a90>
>>> plt.show()
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Tue Dec 12 14:14:43 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
1. - ^ 	: Session  (took  58.47992  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Wed Dec 13 11:02:15 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
2. -- v 	: epoch: 0
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: [784 500] 
Info 		: 	Error 0.0488086 
2. -- ^ 	: epoch: 0  (took  28.23907  sek)
------------------------------------------------
Info 		: Error: 0.0488086 
Info 		: Learnrate: 0.1  // Batchsize: 110  // Temp.: 2.5 
1. - ^ 	: Session  (took  34.30691  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Wed Dec 13 11:03:19 2017 
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
loading from: /Users/Niermann/Google Drive/Masterarbeit/Python
loaded 1 objects from file
2. -- v 	: epoch: 0
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: (784, 500) 
Info 		: 	Error 0.0621324 
2. -- ^ 	: epoch: 0  (took  14.17287  sek)
------------------------------------------------
Info 		: Error: 0.0621324 
Info 		: Learnrate: 0.1  // Batchsize: 550  // Temp.: 2.5 
1. - ^ 	: Session  (took  17.58323  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Wed Dec 13 11:04:03 2017 
2. -- v 	: epoch: 0
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: (?, 784) 
Info 		: 	Error 0.143388 
2. -- ^ 	: epoch: 0  (took  14.40474  sek)
------------------------------------------------
Info 		: Error: 0.143388 
Info 		: Learnrate: 0.1  // Batchsize: 550  // Temp.: 2.5 
1. - ^ 	: Session  (took  16.76118  sek)
>>> batch.shape
(550, 784)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Wed Dec 13 11:06:39 2017 
2. -- v 	: epoch: 0
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
[550 784]
Info 		: 	Error 0.143348 
2. -- ^ 	: epoch: 0  (took  20.48588  sek)
------------------------------------------------
Info 		: Error: 0.143348 
Info 		: Learnrate: 0.1  // Batchsize: 550  // Temp.: 2.5 
1. - ^ 	: Session  (took  25.09505  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Wed Dec 13 11:22:50 2017 
2. -- v 	: epoch: 0
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
[550 500]
Info 		: 	Error 0.143086 
2. -- ^ 	: epoch: 0  (took  14.00115  sek)
------------------------------------------------
Info 		: Error: 0.143086 
Info 		: Learnrate: 0.1  // Batchsize: 550  // Temp.: 2.5 
1. - ^ 	: Session  (took  17.1653  sek)
>>> m=rnd.random([3,2])
>>> v=ones([3,1])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'ones' is not defined
>>> v=np.ones([3,1])
>>> np.dot(v,m)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: shapes (3,1) and (3,2) not aligned: 1 (dim 1) != 3 (dim 0)
>>> np.dot(m,v)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: shapes (3,2) and (3,1) not aligned: 2 (dim 1) != 3 (dim 0)
>>> v=np.ones([1,3])
>>> np.dot(m,v)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: shapes (3,2) and (1,3) not aligned: 2 (dim 1) != 1 (dim 0)
>>> np.dot(v,m)
array([[ 1.799,  0.977]])
>>> v
array([[ 1.,  1.,  1.]])
>>> m
array([[ 0.55 ,  0.089],
       [ 0.747,  0.617],
       [ 0.501,  0.271]])
>>> v=np.ones([1,2])
>>> np.dot(m,v)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: shapes (3,2) and (1,2) not aligned: 2 (dim 1) != 1 (dim 0)
>>> v
array([[ 1.,  1.]])
>>> v=np.ones([2])
>>> np.dot(m,v)
array([ 0.639,  1.365,  0.772])
>>> v
array([ 1.,  1.])
>>> np.random([100])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'module' object is not callable
>>> rnd.random([100])
array([ 0.71 ,  0.67 ,  0.82 ,  0.391,  0.521,  0.327,  0.362,  0.075,
        0.42 ,  0.546,  0.82 ,  0.626,  0.466,  0.021,  0.98 ,  0.939,
        0.83 ,  0.634,  0.55 ,  0.491,  0.059,  0.035,  0.545,  0.769,
        0.429,  0.39 ,  0.885,  0.474,  0.881,  0.991,  0.703,  0.708,
        0.961,  0.389,  0.137,  0.688,  0.044,  0.978,  0.944,  0.537,
        0.559,  0.773,  0.933,  0.714,  0.617,  0.939,  0.031,  0.004,
        0.762,  0.107,  0.285,  0.268,  0.498,  0.633,  0.627,  0.407,
        0.497,  0.872,  0.046,  0.645,  0.368,  0.472,  0.423,  0.29 ,
        0.266,  0.204,  0.476,  0.247,  0.913,  0.285,  0.125,  0.019,
        0.507,  0.974,  0.272,  0.516,  0.039,  0.072,  0.423,  0.082,
        0.899,  0.823,  0.91 ,  0.093,  0.454,  0.585,  0.004,  0.766,
        0.359,  0.725,  0.955,  0.478,  0.736,  0.101,  0.324,  0.305,
        0.158,  0.671,  0.749,  0.668])
>>> rnd.random([1,100])
array([[ 0.383,  0.242,  0.043,  0.626,  0.45 ,  0.983,  0.509,  0.358,
         0.246,  0.014,  0.78 ,  0.042,  0.077,  0.577,  0.307,  0.031,
         0.895,  0.345,  0.052,  0.632,  0.399,  0.792,  0.464,  0.556,
         0.666,  0.909,  0.322,  0.688,  0.067,  0.204,  0.754,  0.028,
         0.864,  0.732,  0.305,  0.308,  0.035,  0.111,  0.885,  0.016,
         0.61 ,  0.459,  0.951,  0.409,  0.429,  0.905,  0.104,  0.458,
         0.016,  0.904,  0.005,  0.62 ,  0.472,  0.393,  0.562,  0.511,
         0.057,  0.067,  0.633,  0.159,  0.496,  0.223,  0.402,  0.361,
         0.934,  0.067,  0.06 ,  0.263,  0.181,  0.355,  0.595,  0.11 ,
         0.814,  0.698,  0.443,  0.116,  0.843,  0.381,  0.855,  0.52 ,
         0.228,  0.681,  0.669,  0.384,  0.149,  0.39 ,  0.499,  0.253,
         0.341,  0.096,  0.33 ,  0.409,  0.05 ,  0.177,  0.548,  0.286,
         0.389,  0.158,  0.223,  0.594]])
>>> rnd.random([2,100])
array([[ 0.092,  0.522,  0.799,  0.995,  0.889,  0.829,  0.185,  0.4  ,
         0.191,  0.135,  0.245,  0.056,  0.296,  0.263,  0.997,  0.537,
         0.579,  0.357,  0.169,  0.216,  0.476,  0.567,  0.365,  0.821,
         0.89 ,  0.782,  0.331,  0.334,  0.482,  0.854,  0.728,  0.394,
         0.816,  0.715,  0.012,  0.103,  0.777,  0.051,  0.071,  0.282,
         0.086,  0.303,  0.135,  0.022,  0.354,  0.739,  0.047,  0.087,
         0.913,  0.011,  0.104,  0.262,  0.031,  0.579,  0.557,  0.682,
         0.8  ,  0.221,  0.248,  0.795,  0.672,  0.355,  0.362,  0.52 ,
         0.373,  0.397,  0.841,  0.005,  0.457,  0.801,  0.072,  0.077,
         0.886,  0.009,  0.097,  0.816,  0.519,  0.616,  0.989,  0.709,
         0.345,  0.168,  0.723,  0.137,  0.996,  0.312,  0.272,  0.628,
         0.951,  0.138,  0.238,  0.404,  0.247,  0.778,  0.762,  0.099,
         0.902,  0.591,  0.953,  0.869],
       [ 0.009,  0.37 ,  0.344,  0.339,  0.998,  0.256,  0.308,  0.823,
         0.19 ,  0.416,  0.256,  0.904,  0.272,  0.711,  0.598,  0.072,
         0.865,  0.563,  0.017,  0.489,  0.772,  0.299,  0.011,  0.476,
         0.529,  0.934,  0.627,  0.133,  0.907,  0.591,  0.875,  0.701,
         0.357,  0.494,  0.582,  0.709,  0.989,  0.225,  0.905,  0.346,
         0.067,  0.051,  0.197,  0.85 ,  0.608,  0.577,  0.794,  0.045,
         0.856,  0.578,  0.295,  0.37 ,  0.665,  0.767,  0.073,  0.147,
         0.69 ,  0.844,  0.411,  0.648,  0.046,  0.369,  0.376,  0.652,
         0.299,  0.686,  0.432,  0.48 ,  0.214,  0.527,  0.983,  0.555,
         0.636,  0.018,  0.038,  0.356,  0.698,  0.775,  0.081,  0.01 ,
         0.288,  0.642,  0.553,  0.232,  0.644,  0.376,  0.341,  0.842,
         0.746,  0.678,  0.262,  0.973,  0.935,  0.937,  0.581,  0.574,
         0.25 ,  0.66 ,  0.234,  0.687]])
>>> rnd.random([2,10])
array([[ 0.883,  0.47 ,  0.05 ,  0.281,  0.168,  0.67 ,  0.825,  0.89 ,
         0.368,  0.793],
       [ 0.987,  0.229,  0.15 ,  0.092,  0.545,  0.086,  0.973,  0.959,
         0.455,  0.711]])
>>> rnd.random([2,5])
array([[ 0.938,  0.734,  0.947,  0.788,  0.578],
       [ 0.812,  0.507,  0.756,  0.271,  0.594]])
>>> rnd.random([10,1])
array([[ 0.546],
       [ 0.04 ],
       [ 0.035],
       [ 0.351],
       [ 0.264],
       [ 0.577],
       [ 0.131],
       [ 0.888],
       [ 0.387],
       [ 0.487]])
>>> rnd.random([10,0])
array([], shape=(10, 0), dtype=float64)
>>> rnd.random([10])
array([ 0.43 ,  0.252,  0.888,  0.603,  0.241,  0.504,  0.114,  0.982,
        0.071,  0.198])
>>> rnd.random([10,1])
array([[ 0.506],
       [ 0.48 ],
       [ 0.94 ],
       [ 0.153],
       [ 0.821],
       [ 0.991],
       [ 0.269],
       [ 0.782],
       [ 0.398],
       [ 0.485]])
>>> np.outer(v,w)
array([[<tf.Tensor 'mul_247:0' shape=(784, 500) dtype=float32>],
       [<tf.Tensor 'mul_248:0' shape=(784, 500) dtype=float32>]], dtype=object)
>>> np.outer(1,1)
array([[1]])
>>> np.outer(1,2)
array([[2]])
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Wed Dec 13 12:19:36 2017 
2. -- v 	: epoch: 0
Info 		: 	Error 0.137902 
2. -- ^ 	: epoch: 0  (took  34.81396  sek)
3. -- v 	: epoch: 1
Info 		: 	Error 0.112297 
3. -- ^ 	: epoch: 1  (took  35.32614  sek)
4. -- v 	: epoch: 2
Info 		: 	Error 0.102129 
4. -- ^ 	: epoch: 2  (took  36.02228  sek)
5. -- v 	: epoch: 3
Info 		: 	Error 0.0930196 
5. -- ^ 	: epoch: 3  (took  35.85219  sek)
6. -- v 	: epoch: 4
Info 		: 	Error 0.0865412 
6. -- ^ 	: epoch: 4  (took  35.83348  sek)
7. -- v 	: epoch: 5
Info 		: 	Error 0.08153 
7. -- ^ 	: epoch: 5  (took  35.09894  sek)
8. -- v 	: epoch: 6
Info 		: 	Error 0.0799946 
8. -- ^ 	: epoch: 6  (took  34.9396  sek)
9. -- v 	: epoch: 7
Info 		: 	Error 0.0744028 
9. -- ^ 	: epoch: 7  (took  35.68028  sek)
10. -- v 	: epoch: 8
Info 		: 	Error 0.0739159 
10. -- ^ 	: epoch: 8  (took  35.95581  sek)
11. -- v 	: epoch: 9
Info 		: 	Error 0.0692294 
11. -- ^ 	: epoch: 9  (took  35.25296  sek)
------------------------------------------------
Info 		: Error: 0.0692294 
Info 		: Learnrate: 0.01  // Batchsize: 55  // Temp.: 2.5 
1. - ^ 	: Session  (took  359.67877  sek)
>>> Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 4, in <module>
NameError: name 'num_batches' is not defined
>>> >>> Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 5, in <module>
NameError: name 'learnrate_max' is not defined
>>> >>> 1.9e-05
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Wed Dec 13 12:28:31 2017 
2. -- v 	: epoch: 0
Info 		: Number_of_updates: 999 
Info 		: Learnrate: 0.0574525 
Info 		: 	Error 0.136772 
2. -- ^ 	: epoch: 0  (took  35.92416  sek)
3. -- v 	: epoch: 1
Info 		: Number_of_updates: 1998 
Info 		: Learnrate: 0.104905 
Info 		: 	Error 0.11047 
3. -- ^ 	: epoch: 1  (took  36.13982  sek)
4. -- v 	: epoch: 2
Info 		: Number_of_updates: 2997 
Info 		: Learnrate: 0.1523575 
Info 		: 	Error 0.100417 
4. -- ^ 	: epoch: 2  (took  35.53997  sek)
5. -- v 	: epoch: 3
Info 		: Number_of_updates: 3996 
Info 		: Learnrate: 0.19981 
Info 		: 	Error 0.0929293 
5. -- ^ 	: epoch: 3  (took  37.9537  sek)
------------------------------------------------
Info 		: Error: 0.0929293 
Info 		: Learnrate: 0.19981  // Batchsize: 55  // Temp.: 2.5 
1. - ^ 	: Session  (took  149.91897  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Wed Dec 13 12:33:46 2017 
2. -- v 	: epoch: 0
Info 		: 	Learnrate: 0.2572525 
Info 		: 	Error 0.139213 
2. -- ^ 	: epoch: 0  (took  37.55745  sek)
3. -- v 	: epoch: 1
Info 		: 	Learnrate: 0.504505 
Info 		: 	Error 0.111242 
3. -- ^ 	: epoch: 1  (took  30.78746  sek)
4. -- v 	: epoch: 2
Info 		: 	Learnrate: 0.7517575 
Info 		: 	Error 0.101412 
4. -- ^ 	: epoch: 2  (took  24.96201  sek)
5. -- v 	: epoch: 3
Info 		: 	Learnrate: 0.99901 
Info 		: 	Error 0.0925565 
5. -- ^ 	: epoch: 3  (took  25.58592  sek)
------------------------------------------------
Info 		: Error: 0.0925565 
Info 		: Learnrate: 0.99901  // Batchsize: 55  // Temp.: 2.5 
1. - ^ 	: Session  (took  123.77605  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Wed Dec 13 12:36:17 2017 
2. -- v 	: epoch: 0
Info 		: 	Learnrate: 0.324775 
Info 		: 	Error 0.0686081 
2. -- ^ 	: epoch: 0  (took  109.15401  sek)
3. -- v 	: epoch: 1
Info 		: 	Learnrate: 0.54955 
Info 		: 	Error 0.055625 
3. -- ^ 	: epoch: 1  (took  56.82907  sek)
4. -- v 	: epoch: 2
Info 		: 	Learnrate: 0.774325 
Info 		: 	Error 0.0503764 
4. -- ^ 	: epoch: 2  (took  56.94242  sek)
5. -- v 	: epoch: 3
Info 		: 	Learnrate: 0.9991 
Info 		: 	Error 0.0474885 
5. -- ^ 	: epoch: 3  (took  54.80343  sek)
------------------------------------------------
Info 		: Error: 0.0474885 
Info 		: Learnrate: 0.9991  // Batchsize: 55  // Temp.: 2.5 
1. - ^ 	: Session  (took  281.26314  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Wed Dec 13 12:41:42 2017 
2. -- v 	: epoch: 0
Info 		: 	Learnrate: 2.872625 
Info 		: 	Error 0.0865312 
2. -- ^ 	: epoch: 0  (took  38.35584  sek)
3. -- v 	: epoch: 1
Info 		: 	Learnrate: 5.24525 
Info 		: 	Error 0.0709495 
3. -- ^ 	: epoch: 1  (took  26.23093  sek)
4. -- v 	: epoch: 2
Info 		: 	Learnrate: 7.617875 
Info 		: 	Error 0.0631313 
4. -- ^ 	: epoch: 2  (took  25.65715  sek)
5. -- v 	: epoch: 3
Info 		: 	Learnrate: 9.9905 
Info 		: 	Error 0.0536128 
5. -- ^ 	: epoch: 3  (took  25.29674  sek)
------------------------------------------------
Info 		: Error: 0.0536128 
Info 		: Learnrate: 9.9905  // Batchsize: 55  // Temp.: 1.5 
1. - ^ 	: Session  (took  118.87889  sek)
>>> plt.plot(errors)
[<matplotlib.lines.Line2D object at 0x1225c8d10>]
>>> plt.show()
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Wed Dec 13 12:44:23 2017 
2. -- v 	: epoch: 0
Info 		: 	Learnrate: 1.00901 
Info 		: 	Error 0.111016 
2. -- ^ 	: epoch: 0  (took  110.95907  sek)
3. -- v 	: epoch: 1
Info 		: 	Learnrate: 1.99802 
Info 		: 	Error 0.0914624 
3. -- ^ 	: epoch: 1  (took  57.79841  sek)
------------------------------------------------
Info 		: Error: 0.0914624 
Info 		: Learnrate: 1.99802  // Batchsize: 55  // Temp.: 2.5 
1. - ^ 	: Session  (took  172.10168  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Wed Dec 13 12:47:54 2017 
2. -- v 	: epoch: 0
Info 		: 	Learnrate: 0.363217857143 
Info 		: 	Error 0.136399 
2. -- ^ 	: epoch: 0  (took  63.37425  sek)
3. -- v 	: epoch: 1
Info 		: 	Learnrate: 0.716435714286 
Info 		: 	Error 0.111363 
3. -- ^ 	: epoch: 1  (took  25.29552  sek)
4. -- v 	: epoch: 2
Info 		: 	Learnrate: 1.06965357143 
Info 		: 	Error 0.101602 
4. -- ^ 	: epoch: 2  (took  25.48912  sek)
5. -- v 	: epoch: 3
Info 		: 	Learnrate: 1.42287142857 
Info 		: 	Error 0.0941314 
5. -- ^ 	: epoch: 3  (took  25.98339  sek)
------------------------------------------------
Info 		: Error: 0.0941314 
Info 		: Learnrate: 1.42287142857  // Batchsize: 55  // Temp.: 2.5 
1. - ^ 	: Session  (took  143.50773  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Wed Dec 13 12:50:35 2017 
2. -- v 	: epoch: 0
Info 		: 	Learnrate: 0.38 
Info 		: 	Error 0.097 
2. -- ^ 	: epoch: 0  (took  76.57973  sek)
3. -- v 	: epoch: 1
Info 		: 	Learnrate: 0.72 
Info 		: 	Error 0.081 
3. -- ^ 	: epoch: 1  (took  26.33733  sek)
4. -- v 	: epoch: 2
Info 		: 	Learnrate: 1.07 
Info 		: 	Error 0.071 
4. -- ^ 	: epoch: 2  (took  24.67139  sek)
5. -- v 	: epoch: 3
Info 		: 	Learnrate: 1.41 
Info 		: 	Error 0.064 
5. -- ^ 	: epoch: 3  (took  25.38977  sek)
------------------------------------------------
Info 		: Error: 0.0640728 
Info 		: Learnrate: 1.41432857143  // Batchsize: 55  // Temp.: 2.5 
1. - ^ 	: Session  (took  156.36081  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Wed Dec 13 14:07:42 2017 
2. -- v 	: epoch: 0
Info 		: 	Learnrate: 0.27 
Info 		: 	Error 0.11 
2. -- ^ 	: epoch: 0  (took  31.59782  sek)
3. -- v 	: epoch: 1
Info 		: 	Learnrate: 0.52 
Info 		: 	Error 0.092 
3. -- ^ 	: epoch: 1  (took  30.50098  sek)
4. -- v 	: epoch: 2
Info 		: 	Learnrate: 0.76 
Info 		: 	Error 0.082 
4. -- ^ 	: epoch: 2  (took  30.49456  sek)
5. -- v 	: epoch: 3
Info 		: 	Learnrate: 1.01 
Info 		: 	Error 0.075 
5. -- ^ 	: epoch: 3  (took  31.0801  sek)
------------------------------------------------
Info 		: Error: 0.074505 
Info 		: Learnrate: 1.0  // Batchsize: 78  // Temp.: 2.5 
1. - ^ 	: Session  (took  130.09461  sek)
>>> 
***Repl Killed***

#############
## RESTARPython 2.7.10 (default, Oct 23 2015, 19:19:21) 
[GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.0.59.5)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> Starting...

1. 	      : Loading Data
Extracting MNIST_data/train-images-idx3-ubyte.gz
Extracting MNIST_data/train-labels-idx1-ubyte.gz
Extracting MNIST_data/t10k-images-idx3-ubyte.gz
Extracting MNIST_data/t10k-labels-idx1-ubyte.gz
------------------------------------------------

1. - v 	: Session
Info 		: Wed Dec 13 14:11:10 2017 
2017-12-13 14:11:10.118826: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.2 instructions, but these are available on your machine and could speed up CPU computations.
2017-12-13 14:11:10.118854: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX instructions, but these are available on your machine and could speed up CPU computations.
2017-12-13 14:11:10.118862: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX2 instructions, but these are available on your machine and could speed up CPU computations.
2017-12-13 14:11:10.118870: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use FMA instructions, but these are available on your machine and could speed up CPU computations.
2. -- v 	: Epoch: 0 / 4
Info 		: 	Learnrate: 0.27 
Info 		: 	Error 0.11 
Error: while end logging
3. --- v 	: Epoch: 1 / 4
Info 		: 	Learnrate: 0.52 
Info 		: 	Error 0.091 
Error: while end logging
4. ---- v 	: Epoch: 2 / 4
Info 		: 	Learnrate: 0.76 
Info 		: 	Error 0.081 
Error: while end logging
5. ----- v 	: Epoch: 3 / 4
Info 		: 	Learnrate: 1.01 
Info 		: 	Error 0.076 
Error: while end logging
------------------------------------------------
Info 		: Error: 0.0763258 
Info 		: Learnrate: 1.0  // Batchsize: 78  // Temp.: 2.5 
Error: while end logging
/Library/Python/2.7/site-packages/matplotlib/collections.py:548: FutureWarning: elementwise comparison failed; returning scalar instead, but in the future will perform elementwise comparison
  if self._edgecolors == 'face':
>>> 
***Repl Killed***

#############
## RESTARPython 2.7.10 (default, Oct 23 2015, 19:19:21) 
[GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.0.59.5)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> Starting...

1. 	      : Loading Data
Extracting MNIST_data/train-images-idx3-ubyte.gz
Extracting MNIST_data/train-labels-idx1-ubyte.gz
Extracting MNIST_data/t10k-images-idx3-ubyte.gz
Extracting MNIST_data/t10k-labels-idx1-ubyte.gz
------------------------------------------------

1. - v 	: Session
Info 		: Wed Dec 13 14:13:53 2017 
2017-12-13 14:13:53.062561: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.2 instructions, but these are available on your machine and could speed up CPU computations.
2017-12-13 14:13:53.062573: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX instructions, but these are available on your machine and could speed up CPU computations.
2017-12-13 14:13:53.062579: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX2 instructions, but these are available on your machine and could speed up CPU computations.
2017-12-13 14:13:53.062586: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use FMA instructions, but these are available on your machine and could speed up CPU computations.
2. -- v 	: Epoch: 0 / 2
Info 		: 	Learnrate: 0.51 
Info 		: 	Error 0.111 
2. -- ^ 	: 	Epoch: 0 / 2  (took  18.2  sek)
3. -- v 	: Epoch: 1 / 2
Info 		: 	Learnrate: 1.0 
Info 		: 	Error 0.09 
3. -- ^ 	: 	Epoch: 1 / 2  (took  21.3  sek)
------------------------------------------------
Info 		: Error: 0.0902753 
Info 		: Learnrate: 1.0  // Batchsize: 110  // Temp.: 2.5 
1. - ^ 	: 	Session  (took  40.1  sek)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 246, in <module>
IndexError: index 8 is out of bounds for axis 0 with size 8
>>> reload(numpy)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'numpy' is not defined
>>> reload(np)
<module 'numpy' from '/Library/Python/2.7/site-packages/numpy/__init__.pyc'>
>>> np.val()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'module' object has no attribute 'val'
>>> np.exp()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: invalid number of arguments
>>> len(rec)
11
>>> Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 4, in <module>
IndexError: index 8 is out of bounds for axis 0 with size 8
>>> Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 3, in <module>
IndexError: index 8 is out of bounds for axis 0 with size 8
>>> len(half_object)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'half_object' is not defined
>>> len(half_images)
11
>>> half_images[8]
array([ 0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.067,  0.184,  0.184,  0.184,  0.063,  0.506,  0.333,
        0.184,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.294,  0.6  ,  0.851,  0.992,  0.992,
        0.992,  0.843,  0.965,  0.992,  0.992,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.137,  0.557,  0.957,  0.988,
        0.992,  0.992,  0.992,  0.992,  0.992,  0.992,  0.992,  0.992,
        0.992,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.247,  0.992,  0.992,  0.992,  0.992,  0.992,  0.992,  0.992,
        0.835,  0.667,  0.667,  0.667,  0.667,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.078,  0.518,  0.282,  0.   ,  0.224,  0.933,  0.89 ,  0.933,
        0.659,  0.486,  0.271,  0.078,  0.043,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.043,  0.808,  0.992,  0.306,  0.   ,
        0.   ,  0.125,  0.   ,  0.118,  0.008,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.024,  0.694,
        0.992,  0.518,  0.039,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.047,  0.522,  0.992,  0.914,  0.059,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.361,  0.992,  0.875,
        0.11 ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.588,  0.992,  0.682,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.918,  0.992,  0.965,
        0.498,  0.192,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  1.   ,  0.992,  0.992,  0.992,  0.984,  0.576,  0.357,
        0.475,  0.333,  0.165,  0.165,  0.333,  0.11 ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.545,  0.992,  0.992,
        0.992,  0.992,  0.992,  0.992,  0.992,  0
.992,  0.992,  0.992,
        0.992,  0.91 ,  0.659,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.012,  0.208,  0.855,  0.871,  0.984,  0.992,  0.992,
        0.992,  0.992,  0.992,  0.992,  0.992,  0.992,  0.988,  0.486,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.263,  0.282,  0.784,  0.992,  0.992,  0.992,  0.992,
        0.992,  0.992,  0.992,  0.686,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.471,
        0.992,  0.976,  0.596,  0.2  ,  0.643,  0.992,  0.992,  0.686,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.196,  0.992,  0.992,  0.992,  0.737,
        0.988,  0.992,  0.992,  0.58 ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.035,
        0.655,  0.992,  0.992,  0.992,  0.992,  0.98 ,  0.686,  0.043,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.09 ,  0.706,  0.906,  0.992,
        0.867,  0.502,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.365,  0.584,  0.086,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,
        0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ,  0.   ], dtype=float32)
>>> >>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Wed Dec 13 14:17:40 2017 
2. -- v 	: Epoch: 0 / 2
Info 		: 	Learnrate: 0.51 
Info 		: 	Error 0.11 
2. -- ^ 	: 	Epoch: 0 / 2  (took  27.0  sek)
3. -- v 	: Epoch: 1 / 2
Info 		: 	Learnrate: 1.0 
Info 		: 	Error 0.09 
3. -- ^ 	: 	Epoch: 1 / 2  (took  26.7  sek)
------------------------------------------------
Info 		: Error: 0.0903439 
Info 		: Learnrate: 1.0  // Batchsize: 110  // Temp.: 2.5 
1. - ^ 	: 	Session  (took  54.4  sek)
/Library/Python/2.7/site-packages/matplotlib/collections.py:548: FutureWarning: elementwise comparison failed; returning scalar instead, but in the future will perform elementwise comparison
  if self._edgecolors == 'face':
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Wed Dec 13 14:22:42 2017 
2. -- v 	: Epoch: 1 / 3
Info 		: 	Learnrate: 0.37 
Info 		: 	Error 0.093 
2. -- ^ 	: 	Epoch: 1 / 3  (took  27.4  sek)
3. -- v 	: Epoch: 2 / 3
Info 		: 	Learnrate: 0.68 
Info 		: 	Error 0.077 
3. -- ^ 	: 	Epoch: 2 / 3  (took  26.8  sek)
4. -- v 	: Epoch: 3 / 3
Info 		: 	Learnrate: 1.0 
Info 		: 	Error 0.069 
4. -- ^ 	: 	Epoch: 3 / 3  (took  26.8  sek)
------------------------------------------------
Info 		: Error: 0.068993 
Info 		: Learnrate: 1.0  // Batchsize: 110  // Temp.: 2.5 
1. - ^ 	: 	Session  (took  81.9  sek)
>>> len(test_data)
10000
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Wed Dec 13 14:26:58 2017 
2. -- v 	: Epoch: 1 / 3
Info 		: 	Learnrate: 0.37 
Info 		: 	Error 0.093 
2. -- ^ 	: 	Epoch: 1 / 3  (took  26.3  sek)
3. -- v 	: Epoch: 2 / 3
Info 		: 	Learnrate: 0.68 
Info 		: 	Error 0.078 
3. -- ^ 	: 	Epoch: 2 / 3  (took  25.9  sek)
4. -- v 	: Epoch: 3 / 3
Info 		: 	Learnrate: 1.0 
Info 		: 	Error 0.067 
4. -- ^ 	: 	Epoch: 3 / 3  (took  25.9  sek)
------------------------------------------------
Info 		: Train Error: 0.0670607 
Info 		: Test Error: [0.07281889] 
Info 		: Learnrate: 1.0  // Batchsize: 110  // Temp.: 2.5 
1. - ^ 	: 	Session  (took  80.0  sek)
>>> w_i.min()
-0.99440825
>>> w_i.mean()
-0.064939156
>>> 
>>> w_i.var()
0.014984678
>>> sqrt(w_i.var())
0.12241191749877256
>>> plt.plot(w_i.flatten())
[<matplotlib.lines.Line2D object at 0x12738d1d0>]
>>> plt.show()
>>> plt.plot(sorted(w_i.flatten()))
[<matplotlib.lines.Line2D object at 0x128d60290>]
>>> plt.show()
2017-12-13 14:40:46.623 Python[57308:16911348] [QL] file:///Users/Niermann/Library/QuickLook/qlImageSize.qlgenerator/ has no identifier: ignoring
>>> 2017-12-13 14:41:44.034 Python[57308:16912079] Communications error: <OS_xpc_error: <error: 0x7fff7c994b90> { count = 1, contents =
	"XPCErrorDescription" => <string: 0x7fff7c994f40> { length = 22, contents = "Connection interrupted" }
}>
plt.plot(w_i.flatten())
[<matplotlib.lines.Line2D object at 0x129a75e50>]
>>> plt.show()
>>> plt.plot(w_i.flatten())
[<matplotlib.lines.Line2D object at 0x129ee4790>]
>>> plt.show()
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Wed Dec 13 14:43:44 2017 
2. -- v 	: Epoch: 1 / 1
Info 		: 	Learnrate: 1.0 
Info 		: 	Error 0.153 
2. -- ^ 	: 	Epoch: 1 / 1  (took  18.9  sek)
------------------------------------------------
Info 		: Train Error: 0.153313 
Info 		: Test Error: 0.152636 
Info 		: Learnrate: 1.0  // Batchsize: 110  // Temp.: 2.5 
1. - ^ 	: 	Session  (took  20.4  sek)
>>> plt.plot(w_i.flatten())
[<matplotlib.lines.Line2D object at 0x11e356790>]
>>> plt.show()
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Wed Dec 13 14:45:34 2017 
2. -- v 	: Epoch: 1 / 1
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<string>", line 190, in <module>
  File "/Library/Python/2.7/site-packages/matplotlib/axes.py", line 9105, in matshow
    nr, nc = Z.shape
ValueError: need more than 1 value to unpack
>>> flat_w
array([-0.003, -0.002, -0.003, ..., -0.003, -0.003, -0.002], dtype=float32)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Wed Dec 13 14:46:02 2017 
2. -- v 	: Epoch: 1 / 1
Info 		: 	Learnrate: 1.0 
Info 		: 	Error 0.153 
2. -- ^ 	: 	Epoch: 1 / 1  (took  37.9  sek)
------------------------------------------------
Info 		: Train Error: 0.152645 
Info 		: Test Error: 0.153063 
Info 		: Learnrate: 1.0  // Batchsize: 110  // Temp.: 2.5 
1. - ^ 	: 	Session  (took  39.9  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Wed Dec 13 14:47:14 2017 
2. -- v 	: Epoch: 1 / 1
Info 		: 	Learnrate: 1.0 
Info 		: 	Error 0.153 
2. -- ^ 	: 	Epoch: 1 / 1  (took  57.8  sek)
------------------------------------------------
Info 		: Train Error: 0.152695 
Info 		: Test Error: 0.153068 
Info 		: Learnrate: 1.0  // Batchsize: 110  // Temp.: 2.5 
1. - ^ 	: 	Session  (took  59.5  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Wed Dec 13 14:48:36 2017 
2. -- v 	: Epoch: 1 / 1
Info 		: 	Learnrate: 1.0 
Info 		: 	Error 0.152 
2. -- ^ 	: 	Epoch: 1 / 1  (took  83.7  sek)
------------------------------------------------
Info 		: Train Error: 0.152298 
Info 		: Test Error: 0.152694 
Info 		: Learnrate: 1.0  // Batchsize: 110  // Temp.: 2.5 
1. - ^ 	: 	Session  (took  85.6  sek)
>>> 2017-12-13 14:53:14.625 Python[57308:16922179] Communications error: <OS_xpc_error: <error: 0x7fff7c994b90> { count = 1, contents =
	"XPCErrorDescription" => <string: 0x7fff7c994f40> { length = 22, contents = "Connection interrupted" }
}>

***Repl Killed***

#############
## RESTARPython 2.7.10 (default, Oct 23 2015, 19:19:21) 
[GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.0.59.5)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> Starting...

1. 	      : Loading Data
Extracting MNIST_data/train-images-idx3-ubyte.gz
Extracting MNIST_data/train-labels-idx1-ubyte.gz
Extracting MNIST_data/t10k-images-idx3-ubyte.gz
Extracting MNIST_data/t10k-labels-idx1-ubyte.gz
------------------------------------------------

1. - v 	: Session
Info 		: Thu Dec 14 15:32:35 2017 
2017-12-14 15:32:35.284715: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.2 instructions, but these are available on your machine and could speed up CPU computations.
2017-12-14 15:32:35.284759: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX instructions, but these are available on your machine and could speed up CPU computations.
2017-12-14 15:32:35.284775: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX2 instructions, but these are available on your machine and could speed up CPU computations.
2017-12-14 15:32:35.284792: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use FMA instructions, but these are available on your machine and could speed up CPU computations.
2. -- v 	: Epoch: 1 / 1
Info 		: 	Learnrate: 1.0 
Info 		: 	Error 0.05 
2. -- ^ 	: 	Epoch: 1 / 1  (took  57.6  sek)
------------------------------------------------
Info 		: Train Error: 0.0498384 
Info 		: Test Error: 0.0537183 
Info 		: Learnrate: 1.0  // Batchsize: 110  // Temp.: 0.05 
1. - ^ 	: 	Session  (took  58.9  sek)
/Library/Python/2.7/site-packages/matplotlib/collections.py:548: FutureWarning: elementwise comparison failed; returning scalar instead, but in the future will perform elementwise comparison
  if self._edgecolors == 'face':
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Thu Dec 14 15:34:06 2017 
2. -- v 	: Epoch: 1 / 1
Info 		: 	Learnrate: 1.0 
Info 		: 	Error 0.168 
2. -- ^ 	: 	Epoch: 1 / 1  (took  35.3  sek)
------------------------------------------------
Info 		: Train Error: 0.168477 
Info 		: Test Error: 0.169237 
Info 		: Learnrate: 1.0  // Batchsize: 110  // Temp.: 10.0 
1. - ^ 	: 	Session  (took  36.6  sek)
>>> plt.matshow(train_data[0:1])
<matplotlib.image.AxesImage object at 0x119402490>
>>> plt.show()
>>> plt.matshow(train_data[0:1].reshape(28,28))
<matplotlib.image.AxesImage object at 0x11bd0ccd0>
>>> plt.show()
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Thu Dec 14 15:36:50 2017 
2. -- v 	: Epoch: 1 / 1
Info 		: 	Learnrate: 1.0 
Info 		: 	Error 0.12 
2. -- ^ 	: 	Epoch: 1 / 1  (took  45.2  sek)
------------------------------------------------
Info 		: Train Error: 0.120139 
Info 		: Test Error: 0.124127 
Info 		: Learnrate: 1.0  // Batchsize: 110  // Temp.: 10.0 
1. - ^ 	: 	Session  (took  46.8  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Thu Dec 14 15:38:23 2017 
2. -- v 	: Epoch: 1 / 10
Info 		: 	Learnrate: 0.12 
Info 		: 	Error 0.159 
2. -- ^ 	: 	Epoch: 1 / 10  (took  18.3  sek)
3. -- v 	: Epoch: 2 / 10
Info 		: 	Learnrate: 0.22 
Info 		: 	Error 0.152 
3. -- ^ 	: 	Epoch: 2 / 10  (took  18.3  sek)
4. -- v 	: Epoch: 3 / 10
Info 		: 	Learnrate: 0.31 
Info 		: 	Error 0.147 
4. -- ^ 	: 	Epoch: 3 / 10  (took  18.2  sek)
5. -- v 	: Epoch: 4 / 10
Info 		: 	Learnrate: 0.41 
Info 		: 	Error 0.14 
5. -- ^ 	: 	Epoch: 4 / 10  (took  18.2  sek)
6. -- v 	: Epoch: 5 / 10
Info 		: 	Learnrate: 0.51 
Info 		: 	Error 0.132 
6. -- ^ 	: 	Epoch: 5 / 10  (took  18.2  sek)
7. -- v 	: Epoch: 6 / 10
Info 		: 	Learnrate: 0.61 
Info 		: 	Error 0.123 
7. -- ^ 	: 	Epoch: 6 / 10  (took  18.2  sek)
8. -- v 	: Epoch: 7 / 10
Info 		: 	Learnrate: 0.7 
Info 		: 	Error 0.115 
8. -- ^ 	: 	Epoch: 7 / 10  (took  18.2  sek)
9. -- v 	: Epoch: 8 / 10
Info 		: 	Learnrate: 0.8 
Info 		: 	Error 0.109 
9. -- ^ 	: 	Epoch: 8 / 10  (took  18.1  sek)
10. -- v 	: Epoch: 9 / 10
Info 		: 	Learnrate: 0.9 
Info 		: 	Error 0.106 
10. -- ^ 	: 	Epoch: 9 / 10  (took  18.2  sek)
11. -- v 	: Epoch: 10 / 10
Info 		: 	Learnrate: 1.0 
Info 		: 	Error 0.102 
11. -- ^ 	: 	Epoch: 10 / 10  (took  18.2  sek)
------------------------------------------------
Info 		: Train Error: 0.102438 
Info 		: Test Error: 0.105869 
Info 		: Learnrate: 1.0  // Batchsize: 110  // Temp.: 10.0 
1. - ^ 	: 	Session  (took  183.3  sek)
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Fri Dec 15 19:18:14 2017 
2. -- v 	: Epoch: 1 / 1

***Repl Killed***

#############
## RESTARPython 2.7.10 (default, Oct 23 2015, 19:19:21) 
[GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.0.59.5)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> Starting...

1. 	      : Loading Data
Extracting MNIST_data/train-images-idx3-ubyte.gz
Extracting MNIST_data/train-labels-idx1-ubyte.gz
Extracting MNIST_data/t10k-images-idx3-ubyte.gz
Extracting MNIST_data/t10k-labels-idx1-ubyte.gz
------------------------------------------------

1. - v 	: Session
Info 		: Fri Dec 15 19:18:46 2017 
2017-12-15 19:18:47.105286: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.2 instructions, but these are available on your machine and could speed up CPU computations.
2017-12-15 19:18:47.105307: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX instructions, but these are available on your machine and could speed up CPU computations.
2017-12-15 19:18:47.105323: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX2 instructions, but these are available on your machine and could speed up CPU computations.
2017-12-15 19:18:47.105329: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use FMA instructions, but these are available on your machine and could speed up CPU computations.
2. -- v 	: Epoch: 1 / 1

***Repl Killed***

#############
## RESTARPython 2.7.10 (default, Oct 23 2015, 19:19:21) 
[GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.0.59.5)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> Starting...

1. 	      : Loading Data
Extracting MNIST_data/train-images-idx3-ubyte.gz
Extracting MNIST_data/train-labels-idx1-ubyte.gz
Extracting MNIST_data/t10k-images-idx3-ubyte.gz
Extracting MNIST_data/t10k-labels-idx1-ubyte.gz
------------------------------------------------

1. - v 	: Session
Info 		: Fri Dec 15 19:19:14 2017 
2017-12-15 19:19:14.469125: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.2 instructions, but these are available on your machine and could speed up CPU computations.
2017-12-15 19:19:14.469157: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX instructions, but these are available on your machine and could speed up CPU computations.
2017-12-15 19:19:14.469163: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX2 instructions, but these are available on your machine and could speed up CPU computations.
2017-12-15 19:19:14.469169: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use FMA instructions, but these are available on your machine and could speed up CPU computations.
2. -- v 	: Epoch: 1 / 1
Info 		: 	Learnrate: 1.0 
Info 		: 	Error 0.058 
2. -- ^ 	: 	Epoch: 1 / 1  (took  59.8  sek)
------------------------------------------------
Info 		: Train Error: 0.0584853 
Info 		: Test Error: 0.0616803 
Info 		: Learnrate: 1.0  // Batchsize: 110  // Temp.: 2.0 
1. - ^ 	: 	Session  (took  61.0  sek)
/Library/Python/2.7/site-packages/matplotlib/collections.py:548: FutureWarning: elementwise comparison failed; returning scalar instead, but in the future will perform elementwise comparison
  if self._edgecolors == 'face':
>>> Starting...
------------------------------------------------

1. - v 	: Session
Info 		: Mon Dec 18 11:17:35 2017 
2. -- v 	: Epoch: 1 / 10
Info 		: 	Learnrate: 0.01 
Info 		: 	Error 0.161 
2. -- ^ 	: 	Epoch: 1 / 10  (took  36.2  sek)
3. -- v 	: Epoch: 2 / 10
Info 		: 	Learnrate: 0.02 
Info 		: 	Error 0.15 
3. -- ^ 	: 	Epoch: 2 / 10  (took  35.4  sek)
4. -- v 	: Epoch: 3 / 10
Info 		: 	Learnrate: 0.03 
Info 		: 	Error 0.151 
4. -- ^ 	: 	Epoch: 3 / 10  (took  35.2  sek)
5. -- v 	: Epoch: 4 / 10
Info 		: 	Learnrate: 0.04 
Info 		: 	Error 0.145 
5. -- ^ 	: 	Epoch: 4 / 10  (took  35.0  sek)
6. -- v 	: Epoch: 5 / 10
Info 		: 	Learnrate: 0.05 
Info 		: 	Error 0.138 
6. -- ^ 	: 	Epoch: 5 / 10  (took  35.1  sek)
7. -- v 	: Epoch: 6 / 10
Info 		: 	Learnrate: 0.06 
Info 		: 	Error 0.13 
7. -- ^ 	: 	Epoch: 6 / 10  (took  35.8  sek)
8. -- v 	: Epoch: 7 / 10
Info 		: 	Learnrate: 0.07 
Info 		: 	Error 0.122 
8. -- ^ 	: 	Epoch: 7 / 10  (took  35.6  sek)
9. -- v 	: Epoch: 8 / 10
Info 		: 	Learnrate: 0.08 
Info 		: 	Error 0.118 
9. -- ^ 	: 	Epoch: 8 / 10  (took  35.6  sek)
10. -- v 	: Epoch: 9 / 10
Info 		: 	Learnrate: 0.09 
Info 		: 	Error 0.115 
10. -- ^ 	: 	Epoch: 9 / 10  (took  36.0  sek)
11. -- v 	: Epoch: 10 / 10
Info 		: 	Learnrate: 0.1 
Info 		: 	Error 0.11 
11. -- ^ 	: 	Epoch: 10 / 10  (took  35.6  sek)
------------------------------------------------
Info 		: Train Error: 0.110038 
Info 		: Test Error: 0.105881 
Info 		: Learnrate: 0.0  // Batchsize: 55  // Temp.: 2.0 
1. - ^ 	: 	Session  (took  357.5  sek)
>>> np.convolve()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: convolve() takes at least 2 arguments (0 given)
>>> np.convolve([1,2,3],[1,2,3])
array([ 1,  4, 10, 12,  9])
>>> np.convolve([1,2,3],[1,2,3],"same")
array([ 4, 10, 12])
>>> np.convolve([1,2,3],[9,2,3])
array([ 9, 20, 34, 12,  9])
>>> np.convolve([1,2,3],[1,1,1])
array([1, 3, 6, 5, 3])
>>> 
***Repl Killed***
